\chapter{Genomics of the transcription elongation factor Spt5}
\label{chapter:five}

\section{Abstract}

Spt5 is a conserved transcription elongation factor important for the processivity of the transcription complex and many transcription-related processes.
To study the requirement for Spt5 \textit{in vivo}, we have applied multiple genomic assays to \textit{Schizosaccharomyces pombe} cells depleted of Spt5.
Our results reveal an accumulation of RNA Pol II over the 5$^\prime$ ends of genes upon Spt5 depletion, and a progressive decrease in transcript abundance towards the 3$^\prime$ ends of genes.
This is consistent with a model in which Spt5 depletion causes Pol II elongation defects and increases early termination.
We also unexpectedly discover that Spt5 depletion causes hundreds of antisense transcripts to be expressed across the genome, primarily initiating from within the first 500 base pairs of genes.

\section{Collaborators}

\begin{description}[align=right, leftmargin=!, labelwidth=5cm, noitemsep]
    \item [Ameet Shetty] generated TSS-seq, MNase-seq, NET-seq,\\RNA-seq, and ChIP-seq libraries
\end{description}

\section{Introduction to Spt5 and Spt5 depletion}

Spt5 is a fundamental component of the transcription elongation complex, with the distinction of being the only RNA polymerase-associated factor known to be conserved across all three domains of life \citep{hartzog2013, werner2012}.
In eukaryotes, Spt5 heterodimerizes with the protein Spt4, forming a complex known as DSIF (DRB sensitivity-inducing factor) \citep{hartzog1998,hirtreiter2010,schwer2009,wada1998}.
In metazoans, phosphorylation of DSIF controls the release of the elongation complex from promoter-proximal pausing, a regulatory transition state between transcription initiation and productive elongation \citep{adelman2012}.

Within the elongation complex, biochemical and structural studies place Spt4/5 near the center of the action \citep{vos2018a, vos2018b, ehara2017, ehara2019}: Spt5 directly interacts with the noncoding strand of DNA \citep{crickard2016,meyer2015}, the nascent RNA \citep{blythe2016, crickard2016,meyer2015}, and the RNA Pol II clamp domain, which sits above the nucleic acid cleft \citep{hirtreiter2010, martinez-rucobo2011, viktorovskaya2011, yamaguchi1999}.
Binding of Spt5 to Pol II is likely to stabilize the elongation complex and enhance its processivity \citep{hirtreiter2010,klein2011,martinez-rucobo2011,baluapuri2019}, consistent with both \textit{in vitro} studies showing that Spt5 reduces pausing of polymerase under nucleotide-limiting conditions \citep{guo2000,wada1998,zhu2007}, and \textit{in vivo} studies showing elongation defects upon Spt4/5 mutation or depletion \citep{diamant2016,kramer2016,liu2012,mason2005,morillon2003,quan2010,rondon2003}.

As it travels with elongating Pol II, Spt5 recruits other factors, including the Rpd3S histone deactylase complex \citep{drouin2010} and mRNA 3$^\prime$-end processing factors \citep{mayer2012, stadelmayer2014, yamamoto2014}.
The recruitment of still other factors to the elongation complex by Spt5 is dependent on the phosphorylation status of the Spt5 C-terminal region (CTR), a domain composed of tandem repeats analogous to the RNA Pol II C-terminal domain.
When unphosphorylated, the Spt5 CTR aids in recruitment of the mRNA capping enzyme \citep{doamekpor2014, doamekpor2015, schneider2010, wen1999}, while when phosphorylated, the Spt5 CTR recruits the Paf1 complex \citep{liu2009, mbogning2013, wier2013, zhou2009}, another complex involved in transcription elongation \citep{vanoss2017}.

Despite the close relationship of Spt5 to transcription and the transcription-associated processes described above, many studies which knocked down Spt5 in zebrafish, mice, and HeLa cells observed only mild changes in transcript levels across the genome \citep{diamant2016b, komori2009, krishnan2008, stanlie2012, fitz2018}, a result that could potentially be explained by inefficient knockdown of Spt5 and/or the lack of a spike-in control, without which it is impossible to observe a global change over the entire genome \citep{chen2016}.
Consistent with this, more recent studies incoporating spike-in controls have observed global decreases in transcript abundances upon Spt4/5 depletion \citep{henriques2018, naguib2019}.

To study the requirement for Spt5 \textit{in vivo}, we use a system for conditionally depleting Spt5 protein in the fission yeast \textit{Schizosaccharomyces pombe}.
In this system, Spt5 is expressed using the thiamine-repressible \textit{nmt81} promoter \citep{basi1993}, and is tagged with an auxin-inducible degron tag \citep{kanke2011}, such that addition of thiamine and auxin to the media results in repression of \textit{spt5\textsuperscript{+}} transcription and specific degradation of Spt5 protein (Figure \ref{fig:five_depletion_diagram}).
For all experiments described in this chapter, Spt5-depleted cells are sampled 4.5 hours after the start of depletion, at which point the levels of Spt5 on chromatin are about 12\% of non-depleted levels, as measured by ChIP-seq of Spt5 (Figure \ref{fig:five_summary_metagenes}, top panel).

\begin{wrapfigure}[9]{r}{3in}
    \begin{tikzpicture}[x=3in, y=1in]
        \draw [line width=2pt] (0,0.1) -- (0.95,0.1);
        \draw [->, line width=3pt] (0.1,0.1) -- (0.1,0.25) -- (0.25,0.25);
        \draw [line width=2pt] (0.175,0.6) -- (0.175, 0.32);
        \draw [line width=2pt] (0.15, 0.32) -- (0.20, 0.32);
        \node at (0.175, 0.7) {\normalsize thiamine};
        \draw [fill=lightpurple, line width=1pt] (0.28, 0) rectangle (0.8, 0.2);
        \node at (0.54, 0.1) {\normalsize \textit{spt5\textsuperscript{+}}};
        \draw [->, line width=2pt] (0.33, 0.3) to [out=90, in=180] (0.5,0.7);
        \draw [fill=lightpurple, line width=1pt] (0.62, 0.7) circle [x radius=0.1, y radius=0.15];
        \node at (0.62, 0.7) {\normalsize Spt5};
        \draw [->, line width=2pt] (0.73, 0.7) to (0.85, 0.7);
        \node at (0.77, 1) {\normalsize auxin};
        \draw [, line width=2pt] (0.77, 0.9) to [out=270, in=180] (0.80,0.7);
        \node at (0.9, 0.7) {\scalebox{2}{$\varnothing$}};
    \end{tikzpicture}
    \caption[Diagram of the dual-shutoff system used to deplete Spt5 from \textit{S. pombe}]{Diagram of the dual-shutoff system used to deplete Spt5 from \textit{S. pombe}. Spt5 is expressed from a thiamine-repressible promoter, and tagged with an auxin-inducible degron tag for specific degradation upon addition of auxin.}
    \label{fig:five_depletion_diagram}
\end{wrapfigure}

Using this system, we assayed various aspects of transcription and chromatin structure across the genomes of Spt5-depleted and non-depleted cells.
The results are presented below, with section \ref{sec:five_pol_ii} and the RNA-seq part of section \ref{sec:five_transcriptome} describing my reanalyses of data published in \citet{shetty2017} prior to my involvement in this project.

\section{RNA Polymerase II in Spt5 depletion}
\label{sec:five_pol_ii}

To examine the effects of Spt5 depletion on transcription, we performed Pol II ChIP-seq and NET-seq of Spt5-depleted and non-depleted cells.
The data from the two assays paint somewhat different pictures of the changes in Pol II status upon Spt5 depletion (Figure \ref{fig:five_summary_metagenes}).
ChIP-seq reports that global levels of Pol II on chromatin in Spt5-depleted cells are roughly one-third that of non-depleted cells, with the Pol II remaining after Spt5 depletion tending to be located towards the 5$^\prime$ ends of genes.
By contrast, NET-seq reports that total elongating Pol II is not depleted to an appreciable degree but is redistributed from the 3$^\prime$ to the 5$^\prime$ ends of genes, with a greater 5$^\prime$ bias than that observed by ChIP-seq.
We interpret this 5$^\prime$ shift of Pol II as reflecting a transcription elongation defect upon Spt5 depletion, consistent with previously reported elongation defects upon Spt4/5 disruption \citep{diamant2016,kramer2016,liu2012,mason2005,morillon2003,quan2010,rondon2003}.

\begin{wrapfigure}[21]{r}{3in}
    \includegraphics[width=3in]{figures/five/five_summary_metagenes.pdf}
    \caption[Average Spt5 ChIP-seq, RNA Pol II ChIP-seq, and sense NET-seq signal over non-overlapping coding genes, from Spt5-depleted and non-depleted cells.]{Average Spt5 ChIP-seq, RNA Pol II ChIP-seq, and sense NET-seq signal in Spt5 non-depleted and depleted cells, over 1989 non-overlapping coding transcripts scaled from TSS to CPS, plus 0.5 kb on both ends. The solid line and shading are the median and inter-quartile range of the mean spike-in normalized coverage over two replicates or one experiment (non-depleted NET-seq), taken in non-overlapping 20 bp bins and standardized per gene.}
    \label{fig:five_summary_metagenes}
\end{wrapfigure}

To learn more about the state of Pol II after Spt5 depletion, we also performed ChIP-seq for two major post-translational modifications of the Pol II C-terminal domain (CTD), namely serine 5 phosphorylation and serine 2 phosphorylation.
Looking at the relative enrichment of these modifications over gene bodies, we see that the CTD of the Pol II remaining at the 5$^\prime$ ends of genes after Spt5 depletion is enriched for phospho-serine 5 and depleted for phospho-serine 2.
This is somewhat expected due to the respective tendencies of phospho-serine 5 and 2 to occur towards the 5$^\prime$ and 3$^\prime$ ends of genes \citep{komarnitsky2000}.
However, the 5$^\prime$ enrichment of phospho-serine 5 seen in Spt5-depleted cells is not observed in non-depleted cells (note the uniformity of relative Ser5P enrichment in non-depleted cells in Figure \ref{fig:five_rnapii_phosphomark_enrichment}).

\begin{figure}
    \includegraphics[width=6in]{figures/five/five_rnapii_phosphomark_enrichment.pdf}
    \caption[Enrichment of RNA Pol II phospho-serine 5 and phospho-serine 2 over non-overlapping coding genes, in Spt5-depleted and non-depleted cells.]{Median standardized RNA Pol II ChIP-seq signal and relative enrichment of Pol II phospho-serine 5 and phospho-serine 2 ChIP-seq signal in Spt5-depleted and non-depleted cells, over 1989 non-overlapping coding genes aligned by non-depleted genic TSS. ChIP-seq coverage is spike-in normalized and input-subtracted, and relative enrichment of Pol II modifications is a normalized log-ratio of modification coverage over Pol II coverage.}
    \label{fig:five_rnapii_phosphomark_enrichment}
\end{figure}

One possible explanation for the apparent discrepancy between the ChIP-seq and NET-seq results is the difference in immunoprecipitation strategy between the two techniques.
The antibody used to pull down Pol II for ChIP-seq was 8WG16, which recognizes the Pol II CTD.
Reports of this antibody's relative affinity for the various CTD phosphoisoforms vary widely across studies in several species \citep{zeitlinger2007}.
It is conceivable that, for \textit{S. pombe}, the 8WG16 antibody might fail to efficiently pull down a 5$^\prime$-biased phosphoisoform of Pol II that would be captured by NET-seq, a technique that should theoretically capture all Pol II phosphoisoforms via FLAG pulldown of the Rpb3 subunit of Pol II.
If this were the case, it could explain the relative lack of Pol II ChIP-seq signal from Spt5 non-depleted cells over the first ~500 base pairs of genes (Figure \ref{fig:five_rnapii_phosphomark_enrichment}).
Furthermore, if the levels of this missing CTD phosphoisoform were elevated in Spt5-depleted cells versus non-depleted cells, this could also explain the difference in Spt5-depleted total Pol II levels on chromatin as observed by ChIP-seq and NET-seq.

This missing CTD phosphoisoform is not likely to be serine 5 phosphorylation, because ChIP-seq of this mark looks very much like ChIP-seq of Pol II (again, note the uniformity of relative Ser5P enrichment in non-depleted cells in Figure \ref{fig:five_rnapii_phosphomark_enrichment}).
One possible candidate is serine 7 phosphorylation, a modification made early in transcription initiation which has been shown in an \textit{in vitro} human system to be a preferred substrate for P-TEFb to carry out subsequent phosphorylation of serine 5 \citep{czudnochowski2012}.

\section{The transcriptome in Spt5 depletion}
\label{sec:five_transcriptome}

Given the transcriptional changes observed after Spt5 depletion, we performed RNA-seq and TSS-seq to further see how the depletion affects steady-state transcript levels.
Changes to the levels of genic transcripts after Spt5 depletion are generally mild, with the median gene being expressed at roughly 68\% of non-depleted levels as measured by RNA-seq with an RNA spike-in, or at 104\% of non-depleted levels as measured by TSS-seq with a cell spike-in (Figure \ref{fig:five_tss_vs_rna}).
The difference in transcript abundance measurements between RNA-seq and TSS-seq can be explained by a change in the distribution of RNA-seq signal over genes in Spt5 depletion: RNA-seq signal is generally reduced over genes, except near the TSS at the very 5$^\prime$ end of genes (Figure \ref{fig:five_rnaseq_metagene}).
We attribute this to defective elongation upon Spt5 depletion, which increases transcriptional pausing, early termination, and the use of intragenic polyadenylation sites, consistent with a previous report in budding yeast \citep{cui2003}.
\begin{figure}[h]
    \centering
    \includegraphics[width=4.5in]{figures/five/five_tss_vs_rna.pdf}
    \caption[Scatterplot of fold-change in Spt5-depleted over non-depleted cells, comparing TSS-seq and RNA-seq.]{Scatterplot of fold-change in Spt5-depleted over non-depleted cells, comparing TSS-seq and RNA-seq. Each dot represents an RNA-seq measurement over an annotated transcript paired with a TSS-seq measurement over a genic peak assigned to the transcript. Fold-changes are regularized fold-change estimates from DESeq2, with size factors determined from ERCC RNA spike-in counts (RNA-seq) or \textit{S. cerevisiae} cell spike-in counts (TSS-seq).}
    \label{fig:five_tss_vs_rna}
\end{figure}

RNA-seq and TSS-seq also revealed the expression of many novel transcripts in Spt5 depletion, including over 900 antisense transcripts which tend to initiate within the first 500 base pairs downstream of the genic TSS (Figures \ref{fig:five_rnaseq_heatmaps}, \ref{fig:five_antisense_heatmaps}, \ref{fig:five_tss_diffexp_summary}).
Unlike Spt6-repressed transcripts, which are biased towards being sense strand intragenic transcripts (Figure \ref{fig:six_tss_diffexp_summary}), most Spt5-repressed transcripts are biased towards being antisense transcripts (Figure \ref{fig:five_tss_diffexp_summary}).
In general, these Spt5-repressed antisense transcripts are only a few hundred nucleotides in length (Figure \ref{fig:five_antisense_heatmaps}), and are expressed at a lower level than canonical genic transcripts (Figure \ref{fig:five_tss_expression_levels}).
We also find no notable correlation upon Spt5 depletion between changes in antisense transcription and changes in overlapping genic transcription.
Interestingly, the most significant motif found by \textit{de novo} motif discovery upstream of Spt5-repressed antisense TSSs is a GA-rich motif with 3-nucleotide periodicity, similar to the most significant motif found upstream of Spt6-repressed intragenic and antisense TSSs in \textit{S. cerevisiae} (Figures \ref{fig:five_meme_motifs}, \ref{fig:six_meme_motifs}).

\begin{SCfigure}[50][h]
    % \centering
    \includegraphics[width=3.75in]{figures/five/five_rnaseq_metagene.pdf}
    \caption[Average sense RNA-seq signal over non-overlapping coding genes, from Spt5-depleted and non-depleted cells.]{Average sense RNA-seq signal in Spt5 non-depleted and depleted cells, over 1989 non-overlapping coding genes scaled from TSS to CPS. The solid line and shading are the median and inter-quartile range of the mean spike-in normalized coverage over two replicates, taken in non-overlapping 20 nt bins and standardized per gene.}
    \label{fig:five_rnaseq_metagene}
\end{SCfigure}

\begin{figure}[h]
    \centering
    \includegraphics[width=5in]{figures/five/five_rnaseq_heatmaps.pdf}
    \caption[Heatmaps of antisense RNA-seq signal from Spt5-depleted and non-depleted cells, over non-overlapping coding genes.]{Heatmaps of antisense RNA-seq signal from Spt5-depleted and non-depleted cells, over 1989 non-overlapping coding genes aligned by non-depleted genic TSS and sorted by annotated sense transcript length. Data are shown for each gene up to 300 nucleotides 3$^\prime$ of the CPS, indicated by the white dotted line. Values are the mean of spike-in normalized coverage in non-overlapping 20 nucleotide bins, averaged over two replicates. Values above the 93\textsuperscript{rd} percentile are set to the 93\textsuperscript{rd} percentile for visualization.}
    \label{fig:five_rnaseq_heatmaps}
\end{figure}

\clearpage

% \begin{figure}[h]
\begin{sidewaysfigure}
    \centering
    % \includegraphics[width=6in]{figures/five/five_antisense_heatmaps.pdf}
    \includegraphics[width=8.25in]{figures/five/five_antisense_heatmaps.pdf}
    \caption[Heatmaps of antisense TSS-seq, RNA-seq, and NET-seq signal from Spt5-depleted and non-depleted cells, for genes with Spt5-depletion-induced antisense TSSs.]{Heatmaps of antisense TSS-seq, RNA-seq, and NET-seq signal in Spt5 non-depleted and depleted cells, for all genes overlapping an Spt5-depletion-induced antisense TSS. Genes are aligned by the sense, genic TSS and sorted by the distance from the genic TSS to the antisense TSS. Values are the mean of spike-in normalized coverage in non-overlapping 20 nt bins, over one (non-depleted NET-seq) or more experiments. Values above the 0.995 (TSS-seq), 0.98 (RNA-seq), or 0.96 (NET-seq) quantiles are set to their respective quantiles for visualization.}
    \label{fig:five_antisense_heatmaps}
% \end{figure}
\end{sidewaysfigure}

\begin{figure}[h]
    \centering
    \begin{minipage}[t]{2.875in}
        \centering
        \includegraphics[width=2.875in]{figures/five/five_tss_diffexp_summary.pdf}
        \caption[Bar plot of the number of TSS-seq peaks in various genomic classes detected as differentially expressed in Spt5-depleted versus non-depleted cells.]{Bar plot of the number of TSS-seq peaks detected as differentially expressed in Spt5-depleted versus non-depleted cells. The height of each bar is proportional to the total number of peaks in the category, including those not found to be significantly differentially expressed.}
        \label{fig:five_tss_diffexp_summary}
    \end{minipage}\hfill
    \begin{minipage}[t]{2.875in}
        \centering
        \includegraphics[width=2.875in]{figures/five/five_tss_expression_levels.pdf}
        \caption[Violin plots of expression level distributions for genomic classes of TSS-seq peaks in Spt5-depleted and non-depleted cells.]{Violin plots of expression level distributions for genomic classes of TSS-seq peaks in Spt5-depleted and non-depleted cells. Normalized counts are the mean of spike-in size factor normalized counts from four (non-depleted) or two (depleted) replicates.}
        \label{fig:five_tss_expression_levels}
    \end{minipage}
\end{figure}

\begin{figure}[h]
    \centering
    \includegraphics[width=6in]{figures/five/five_meme_motifs.pdf}
    \caption[Sequence logos of motifs discovered by MEME upstream of Spt5-depletion-induced antisense TSSs.]{Sequence logos of motifs discovered by MEME \citep{bailey2015} in the window -100 to +30 bp relative to Spt5-depletion-induced antisense TSSs. For each motif, the observed number of occurrences and the expected number of occurrences if the input sequences were scrambled are shown.}
    \label{fig:five_meme_motifs}
\end{figure}

\clearpage

\section{The chromatin landscape in Spt5 depletion}

\begin{wrapfigure}[13]{r}{3in}
    \includegraphics[width=3in]{figures/five/five_mnase_metagene.pdf}
    \caption[Average MNase-seq dyad signal from Spt5-depleted and non-depleted cells, over non-overlapping coding genes.]{Average MNase-seq dyad signal from Spt5-depleted and non-depleted cells, over 1989 non-overlapping coding genes aligned by wild-type +1 nucleosome dyad. The solid line and shading are the median and inter-quartile range of the mean library-size normalized coverage over two (non-depleted) or three (depleted) replicates.}
    \label{fig:five_mnase_metagene}
\end{wrapfigure}

One hypothesis for why antisense transcripts are expressed upon Spt5 depletion is that changes in chromatin structure create an environment permissive for transcription initiation.
To observe possible changes to chromatin, we performed MNase-seq of Spt5-depleted and non-depleted cells (Figure \ref{fig:five_mnase_metagene}).
Because no spike-in control was included in the experiment, we were unable to use the data to quantify nucleosome occupancy; however, the data do indicate that nucleosomes generally become less well-positioned upon Spt5 depletion (Figure \ref{fig:five_nuc_fuzz}), and that the severity of these changes increases as one moves downstream from the +1 nucleosome into gene bodies (Figure \ref{fig:five_mnase_metagene}).

% \begin{SCfigure}[50][h]
\begin{figure}[h]
    \centering
    \includegraphics[width=6in]{figures/five/five_nuc_fuzz.pdf}
    \caption[Distributions of nucleosome fuzziness in Spt5-depleted and non-depleted cells.]{Distributions of nucleosome fuzziness in Spt5-depleted and non-depleted cells, quantified by DANPOS2 \citep{chen2013}.}
    \label{fig:five_nuc_fuzz}
% \end{SCfigure}
\end{figure}

The data also indicate that Spt5-repressed antisense TSSs generally occur in between the positions of nucleosome dyads, even when viewed as a group (Figure \ref{fig:five_antisense_mnase_metagene}).
Given the tendency of these TSSs to initiate within 500 base pairs downstream of the genic TSS, this is consistent with these TSSs occurring between the +1 and +2, +2 and +3, or +3 and +4 nucleosomes.
Since \textit{S. pombe} nucleosomes have a preference for DNA with low GC content (\citet{moyle-heyrman2013}; left column of Figure \ref{fig:five_antisense_mnase_metagene}) and Spt5-repressed TSSs tend to occur between nucleosome positions, we observe an expected increase in GC content at Spt5-repressed antisense TSSs compared to the surrounding sequence.

\begin{figure}[h]
    \includegraphics[width=6in]{figures/five/five_antisense_mnase_metagene.pdf}
    \caption[Average MNase-seq dyad signal and GC content in Spt5-depleted and non-depleted cells, flanking all antisense TSSs upregulated in Spt5-depleted cells, as well as all genic TSSs detected in non-depleted cells.]{Average MNase-seq dyad signal and GC content of DNA in a 21 bp window for Spt5-depletion-induced antisense TSSs (left panels), as well as all genic TSSs detected in non-depleted cells (right panels). Arrows indicate the direction of transcription for each group of TSSs. The solid line and shading are the median and inter-quartile range of the mean library-size normalized dyad coverage, over two (non-depleted) or three (depleted) replicates, in non-overlapping 10 bp bins.
}
    \label{fig:five_antisense_mnase_metagene}
\end{figure}

We do not observe a systematic change in MNase-seq signal around these TSSs upon Spt5 depletion (Figure \ref{fig:five_antisense_mnase_metagene}), suggesting that Spt5-repressed antisense TSSs probably do not occur as a result of obvious changes to surrounding nucleosomes.
However, it is possible that the increased fuzziness in nucleosome positions upon Spt5 depletion contributes to antisense initiation by creating a chromatin environment favorable for transcription initiation in a subset of the population.
We are also unable to rule out a wholesale decrease in nucleosome occupancy after Spt5 depletion, again owing to the lack of a spike-in control.

\section{Discussion}

In this work, we integrated multiple quantitative genomic approaches to study the conserved transcription elongation factor Spt5.
Our NET-seq and Pol II ChIP-seq results show that, upon Spt5 depletion, Pol II becomes 'stuck' genome-wide at the 5$^\prime$ ends of genes, consistent with the role of Spt5 in stabilizing and enhancing the processivity of the elongation complex.
By TSS-seq and RNA-seq, we see that Spt5 depletion causes mild decreases in steady state RNA signal over gene bodies, but not near the TSS.
This is consistent with a model in which a decrease in elongation complex processivity upon Spt5 depletion causes increased pausing of the elongation complex, early termination, and the use of intragenic polyadenylation signals.
Our transcriptomic assays also unexpectedly revealed that Spt5 depletion leads to the low-level expression of hundreds of antisense transcripts, primarily initiating within the first 500 base pairs downstream of genic TSSs.
To determine if the expression of these antisense transcripts is due to changes in chromatin structure, we performed MNase-seq on Spt5-depleted and non-depleted cells, finding that the antisense transcripts initiate from regions that are already between nucleosomes in non-depleted cells.
The full mechanism of how Spt5 normally represses these transcripts remains to be determined, perhaps involving histone modifications or factors recruited to the elongation complex by Spt5.

\section{Methods}

\subsection{Yeast strain construction and growth conditions}

\textit{S. pombe} strain construction methods are detailed in \citet{shetty2017}.
Spt5 depletion was carried out as follows: Cells were grown in EMM at 30\textdegree C to a density of approximately $1 \times 10^7$ cells/ml ($\text{OD}_{600} \sim 0.5$), at which point thiamine hydrochloride and napthaleneacetic acid (NAA) were added to final concentrations of 100 nM and 0.5 mM, respectively.
The cultures were then incubated with shaking for 4.5 hours at 30\textdegree C.

\subsection{Sequencing library preparation\\(ChIP-seq, NET-seq, RNA-seq, TSS-seq, MNase-seq)}

Library preparation methods for ChIP-seq, NET-seq, and RNA-seq are detailed in \citet{shetty2017}.
TSS-seq and MNase-seq libraries were prepared as described in \citet{doris2018}, except the experimental species was \textit{S. pombe} and the spike-in species for TSS-seq was \textit{S. cerevisiae}.
No spike-in was included in the MNase-seq libraries.

\subsection{Genome builds}

The genome build used for \textit{S. pombe} was ASM294v2 \citep{wood2002}, and the genome build used for \textit{S. cerevisiae} was R64-2-1 \citep{engel2014}.

\subsection{NET-seq data analysis}

NET-seq data analysis was performed as described in section \ref{subsec:net_seq}, except PCR duplicates were removed using a random hexamer molecular barcode present in the adapter, and spike-in normalization was performed by normalizing to the total number of uniquely mapping, non-duplicate \textit{S. cerevisiae} alignments.

\subsection{RNA-seq data analysis}

RNA-seq data analysis was performed using the Snakemake workflow for NET- and RNA-seq analysis described in section \ref{subsec:net_seq}, with the sequences of the ERCC92 synthetic spike-in mix (Thermo Fisher Scientific) as the spike-in genome.
No PCR duplicate removal was performed because no molecular barcode was included in the adapter.

\subsection{ChIP-seq data analysis}

An up-to-date version of the Snakemake \citep{koster2012} workflow used to demultiplex single-end ChIP-seq libraries is maintained at \href{https://github.com/winston-lab/demultiplex-single-end}{github.com/winston-\\lab/demultiplex-single-end}.
At the time of writing, demultiplexing was performed using fastq-multx \citep{aronesty2013}, allowing one mismatch to the barcode.

An up-to-date version of the Snakemake \citep{koster2012} workflow used to process ChIP-seq libraries is maintained at \href{https://github.com/winston-lab/chip-seq}{github.com/winston-lab/chip-seq}.
At the time of writing, 3$^\prime$ quality trimming was performed using cutadapt \citep{martin2011}.
Reads were aligned to the combined \textit{S. pombe} and \textit{S. cerevisiae} genome using Bowtie 2 \citep{langmead2012}, and uniquely mapping alignments were selected using SAMtools \citep{li2009}.
The median fragment size estimated by MACS2 \citep{zhang2008} over all samples was used to generate coverage of factor protection and fragment midpoints by extending reads to the fragment size, or by shifting reads by half the fragment size, respectively.
Input and spike-in normalization was carried out \hyperref[subsubsec:chip_spikein]{as described below}.
Quality statistics of raw, cleaned, non-aligning, and uniquely aligning reads were assessed using FastQC \citep{andrews2010}.

\subsubsection[A note on spike-in normalization for ChIP-seq\\ experiments with input samples]{A note on spike-in normalization for ChIP-seq experiments with input samples}
\label{subsubsec:chip_spikein}

While determining how to do spike-in normalization for ChIP-seq experiments with input samples, I discovered the following error in a published spike-in normalization method.
Throughout the following explanation, I use `experimental' and `spike-in' to refer to the two genomes present in the experiment.

The goal when including spike-ins in a ChIP-seq experiment is to be able to normalize the experimental signal, such that the normalized signal is proportional to the absolute abundance of the factor being immunoprecipitated.
A straightforward method to accomplish this normalization is to linearly scale the experimental signal of a library by a normalization factor, which we will call $\alpha$.
To calculate $\alpha$ for each library, we can use the fact that a normalized `spike-in signal' should be the same for all libraries, since the biological state of the spike-in cells is the same in all libraries.
The key to correctly determining $\alpha$ is defining exactly what this spike-in signal is.

The measurement we begin with for determination of the spike-in signal of a library is the number of reads in the library which map uniquely to the spike-in genome, $R_{\text{spike}}$.
This value will vary based on two factors: the sequencing depth of the library, and the proportion of cells which were spike-in cells, $\phi$:

\begin{align*}
    R_{\text{spike}} &\equiv \text{the number of reads in the library mapping uniquely to the spike-in genome}; \\
    \phi &\equiv \text{the proportion of spike-in cells in the sample}.
\end{align*}
As the derivation of $\alpha$ is more easily understood in terms of absolute cell numbers rather than $\phi$, we will also define the following variables:
\begin{align*}
    C_{\text{exp}} &\equiv \text{the number of experimental cells used to prepare a library}; \\
    C_{\text{spike}} &\equiv \text{the number of spike-in cells used to prepare a library}.
\end{align*}

We can express the \textbf{number of spike-in reads per spike-in cell} by simply taking the fraction $\frac{R_{\text{spike}}}{C_{\text{spike}}}$.
We know that the biological state of a spike-in cell is the same regardless of which sample it belongs to, so $\frac{R_{\text{spike}}}{C_{\text{spike}}}$ is a good candidate for the `spike-in signal' with which to calculate $\alpha$.
However, this expression does not account for differences in $\phi$ between samples: We want two libraries representing the same condition and sequenced to the same depth to have equivalent values of spike-in signal, but this does not hold true for $\frac{R_{\text{spike}}}{C_{\text{spike}}}$ if the two libraries differed in the proportion of spike-in added.

The expression for `spike-in signal' that leads to the correct expression for $\alpha$ is the \textbf{number of spike-in reads per spike-in cell \textit{per experimental cell}}:
\begin{align*}
    & \frac{ \frac{R_\text{spike}}{C_\text{spike}}}{C_\text{exp}} \\
    = & \frac{R_\text{spike} C_\text{exp}}{C_\text{spike}}.
\end{align*}
From here, it's simple to calculate $\alpha$ by setting this value to be equal for all samples.
Since the actual value of the spike-in signal doesn't matter as long as it is equal for all libraries, we can arbitrarily set it to $1$ for convenience:
\begin{align*}
    \alpha \frac{R_\text{spike} C_\text{exp}}{C_\text{spike}} &= 1 \\
    \alpha &= \frac{C_\text{spike}}{R_\text{spike} C_\text{exp}}.
\end{align*}
Notice that only the ratio of spike-in to experimental cells is needed to calculate $\alpha$, and not the absolute number of spike-in and experimental cells.
We can rewrite this expression in terms of $\phi$, the proportion of the sample that was spike-in cells:
\begin{align*}
    \phi &= \frac{C_\text{spike}}{C_\text{spike} + C_\text{exp}} \\
    C_\text{spike} & = \phi \left(C_\text{spike} + C_\text{exp} \right) \\
    C_\text{spike} \left(1-\phi \right) & = \phi C_\text{exp} \\
    \frac{C_\text{spike}}{C_\text{exp}} & = \frac{\phi}{1-\phi} & \alpha &= \frac{C_\text{spike}}{R_\text{spike} C_\text{exp}} \\
                                        && \alpha &= \frac{\phi}{R_\text{spike} \left(1-\phi \right)}.
\end{align*}
This form for $\alpha$ differs from the one presented in \citet{orlando2014} with no derivation:
\begin{align*}
    \alpha &= \frac{\phi}{R_\text{spike} \left(1-\phi \right)} & \alpha_\text{orlando} &= \frac{\phi}{R_\text{spike}}.
\end{align*}
Working through a few examples with both versions of $\alpha$ reveals that using $\alpha_\text{orlando}$ leads to incorrect normalization when $\phi$ is not equivalent for all samples.

In the first example, we will vary sequencing depth between two libraries, keeping everything else constant.
Consider a single ChIP library prep in which 20\% of the cells were spike-in cells (i.e., $\phi=0.2$).
The library is then unevenly split into two aliquots and sequenced.
Library two has four times the reads of library one.
\begin{align*}
    R_{\text{spike}_1} &= 1 & R_{\text{spike}_2} &= 4 \\
    R_{\text{exp}_1} &= 4 & R_{\text{exp}_2} &= 16 \\
\end{align*}
\begin{align*}
    \alpha_1 &= \frac{\phi}{R_{\text{spike}_1} \left(1-\phi \right)} &
    \alpha_2 &= \frac{\phi}{R_{\text{spike}_2} \left(1-\phi \right)} &
    \alpha_{\text{orlando}_1} &= \frac{\phi}{R_{\text{spike}_1}} &
    \alpha_{\text{orlando}_2} &= \frac{\phi}{R_{\text{spike}_2}} \\
    \alpha_1 &= \frac{0.2}{1 \left(0.8 \right)} &
    \alpha_2 &= \frac{0.2}{4 \left(0.8 \right)} &
    \alpha_{\text{orlando}_1} &= \frac{0.2}{1} &
    \alpha_{\text{orlando}_2} &= \frac{0.2}{4} \\
    \alpha_1 &= \frac{4}{16} &
    \alpha_2 &= \frac{1}{16} &
    \alpha_{\text{orlando}_1} &= \frac{4}{20} &
    \alpha_{\text{orlando}_2} &= \frac{1}{20}.
\end{align*}
The total levels of spike-in normalized experimental signal can be found for each library by multiplying $\alpha$ by $R_\text{exp}$, for our version of $\alpha$,
\begin{align*}
    \text{signal}_1 &= \alpha_1 R_{\text{exp}_1}  &
    \text{signal}_2 &= \alpha_2 R_{\text{exp}_2}  \\
    \text{signal}_1 &=  \frac{4}{16} \left(4 \right)  &
    \text{signal}_2 &=  \frac{1}{16} \left(16 \right)  \\
    \text{signal}_1 &=  1 &
    \text{signal}_2 &=  1
\end{align*}
and for $\alpha_\text{orlando}$:
\begin{align*}
    \text{signal}_{\text{orlando}_1} &= \alpha_{\text{orlando}_1} R_{\text{exp}_1} &
    \text{signal}_{\text{orlando}_2} &= \alpha_{\text{orlando}_2} R_{\text{exp}_2} \\
    \text{signal}_{\text{orlando}_1} &= \frac{4}{20} \left(4\right) &
    \text{signal}_{\text{orlando}_2} &= \frac{1}{20} \left(16\right) \\
    \text{signal}_{\text{orlando}_1} &= 0.8 &
    \text{signal}_{\text{orlando}_2} &= 0.8.
\end{align*}
Only the relative abundances within normalization methods matter, so in this case both calculations correctly normalize for library size and say that the normalized signal in the two libraries are the same.

Now consider two libraries from two different conditions with $\phi=0.1$.
In condition 2, a global decrease in experimental signal is expected.
This time, we will skip the algebra:
\begin{align*}
    R_{\text{spike}_1} &= 1 & R_{\text{spike}_2} &= 4 \\
    R_{\text{exp}_1} &= 9 & R_{\text{exp}_2} &= 6
\end{align*}
\begin{align*}
    % \alpha_1 &= \frac{\phi}{R_{\text{spike}_1} \left(1-\phi \right)} &
    % \alpha_2 &= \frac{\phi}{R_{\text{spike}_2} \left(1-\phi \right)} &
    % \alpha_{\text{orlando}_1} &= \frac{\phi}{R_{\text{spike}_1}} &
    % \alpha_{\text{orlando}_2} &= \frac{\phi}{R_{\text{spike}_2}} \\
    % \alpha_1 &= \frac{0.1}{1 \left(0.9 \right)} &
    % \alpha_2 &= \frac{0.1}{4 \left(0.9 \right)} &
    % \alpha_{\text{orlando}_1} &= \frac{0.1}{1} &
    % \alpha_{\text{orlando}_2} &= \frac{0.1}{4} \\
    \alpha_1 &= \frac{4}{36} &
    \alpha_2 &= \frac{1}{36} &
    \alpha_{\text{orlando}_1} &= \frac{4}{40} &
    \alpha_{\text{orlando}_2} &= \frac{1}{40}
\end{align*}
\begin{align*}
    % \text{signal}_1 &= \alpha_1 R_{\text{exp}_1}  &
    % \text{signal}_2 &= \alpha_2 R_{\text{exp}_2}  &
    % \text{signal}_{\text{orlando}_1} &= \alpha_{\text{orlando}_1} R_{\text{exp}_1} &
    % \text{signal}_{\text{orlando}_2} &= \alpha_{\text{orlando}_2} R_{\text{exp}_2} & \\
    % \text{signal}_1 &=  \frac{4}{36} \left(9 \right)  &
    % \text{signal}_2 &=  \frac{1}{36} \left(6 \right)  &
    % \text{signal}_{\text{orlando}_1} &= \frac{4}{40} \left(9\right) &
    % \text{signal}_{\text{orlando}_1} &= \frac{1}{40} \left(6\right) & \\
    \text{signal}_1 &=  1 &
    \text{signal}_2 &=  1/6 &
    \text{signal}_{\text{orlando}_1} &= 0.9 &
    \text{signal}_{\text{orlando}_2} &= 0.15 &
\end{align*}

Both methods correctly detect that experimental signal levels in library two are 1/6th that of library one.

Finally, consider two libraries from the same condition which were spiked in with different amounts of spike-in cells.
Both libraries are sequenced to the same depth.
Since the libraries are from the same condition, we expect their total experimental signal to be the same after normalization, even though they had different amounts of spike-in added.
\begin{align*}
    \phi_1 &= 0.2 & \phi_2 &=0.4 \\
    R_{\text{spike}_1} &= 2 & R_{\text{spike}_2} &= 4 \\
    R_{\text{exp}_1} &= 8 & R_{\text{exp}_2} &= 6 \\
\end{align*}
\begin{align*}
    \alpha_1 &= \frac{\phi_1}{R_{\text{spike}_1} \left(1-\phi_1 \right)} &
    \alpha_2 &= \frac{\phi_2}{R_{\text{spike}_2} \left(1-\phi_2 \right)} &
    \alpha_{\text{orlando}_1} &= \frac{\phi_1}{R_{\text{spike}_1}} &
    \alpha_{\text{orlando}_2} &= \frac{\phi_2}{R_{\text{spike}_2}} \\
    \alpha_1 &= \frac{0.2}{2 \left(0.8 \right)} &
    \alpha_2 &= \frac{0.4}{4 \left(0.6 \right)} &
    \alpha_{\text{orlando}_1} &= \frac{0.2}{2} &
    \alpha_{\text{orlando}_2} &= \frac{0.4}{4} \\
    \alpha_1 &= \frac{3}{24} &
    \alpha_2 &= \frac{4}{24} &
    \alpha_{\text{orlando}_1} &= \frac{1}{10} &
    \alpha_{\text{orlando}_2} &= \frac{1}{10}
\end{align*}
\begin{align*}
    \text{signal}_1 &= \alpha_1 R_{\text{exp}_1}  &
    \text{signal}_2 &= \alpha_2 R_{\text{exp}_2}  \\
    \text{signal}_1 &=  \frac{3}{24} \left(8 \right)  &
    \text{signal}_2 &=  \frac{4}{24} \left(6 \right)  \\
    \text{signal}_1 &=  1 &
    \text{signal}_2 &=  1
\end{align*}
\begin{align*}
    \text{signal}_{\text{orlando}_1} &= \alpha_{\text{orlando}_1} R_{\text{exp}_1} &
    \text{signal}_{\text{orlando}_2} &= \alpha_{\text{orlando}_2} R_{\text{exp}_2} & \\
    \text{signal}_{\text{orlando}_1} &= \frac{1}{10} \left(8\right) &
    \text{signal}_{\text{orlando}_2} &= \frac{1}{10} \left(6\right) & \\
    \text{signal}_{\text{orlando}_1} &= 0.8 &
    \text{signal}_{\text{orlando}_2} &= 0.6
\end{align*}

Here, our method correctly normalizes the two samples to the same total experimental signal while using the Orlando $\alpha$ results in an apparent decrease in signal in library two.
This is because the Orlando $\alpha$ fails to account for the fact that when you add more spike-in to a sample, you necessarily decrease the proportion of the sample that is experimental.
In most experiments with spike-ins, this isn't an issue because we assume that $\phi$ is the same for all samples.
However, for ChIP-seq experiments that include input samples, if we assume that the experimental and spike-in input sample read counts are proportional to the amounts of experimental and spike-in cells, we can plug these values in for values of $\phi$ to get a more reliable estimation of experimental signal levels.
In this case, it becomes important to use the correct equation for $\alpha$.

So, putting everything together, here's how I use a spike-in control to normalize an IP ChIP-seq library paired with an input ChIP-seq library.

As stated above, we assume that the experimental and spike-in read counts in the input sample are proportional to the numbers of experimental and spike-in cells used to prepare the library (In practice, the spike-in may be added as chromatin rather than cells. If the amount of spike-in chromatin added is calculated based on the number of experimental and spike-in cells, as for the ChIP-seq libraries described in this chapter, the following relations should still hold.):
\begin{align*}
    R_{\text{input}_\text{exp}} \propto C_\text{exp}, \\
    R_{\text{input}_\text{spike}} \propto C_\text{spike}.
\end{align*}
Therefore, we can plug these values in for $C$ for both the input and IP libraries (using the form of $\alpha$ without $\phi$):
\begin{align*}
    \alpha_\text{input} &= \frac{C_{\text{input}_\text{spike}}}{R_{\text{input}_\text{spike}} C_{\text{input}_\text{exp}}} &
    \alpha_\text{IP} &= \frac{C_{\text{input}_\text{spike}}}{R_{\text{IP}_\text{spike}} C_{\text{input}_\text{exp}}} \\
    \alpha_\text{input} &\propto \frac{R_{\text{input}_\text{spike}}}{R_{\text{input}_\text{spike}} R_{\text{input}_\text{exp}}} &
    \alpha_\text{IP} &\propto \frac{R_{\text{input}_\text{spike}}}{R_{\text{IP}_\text{spike}} R_{\text{input}_\text{exp}}} \\
    \alpha_\text{input} &\propto \frac{1}{R_{\text{input}_\text{exp}}} &
\end{align*}
Notice how $\alpha_\text{input}$ reduces down to normalizing by the experimental library size, with no dependence on the spike-in.
This makes sense because the input always represents the same state, regardless of how much spike-in is added to it.
The function of the spike-in in the input is only to allow us to estimate abundances in the corresponding IP library.
Rewriting $\alpha_\text{IP}$ in the form
\begin{align*}
    \alpha_\text{IP} &\propto \frac{1}{R_{\text{IP}_\text{spike}} \frac{R_{\text{input}_\text{exp}}}{R_{\text{input}_\text{spike}}}}
\end{align*}
shows that $\alpha_\text{IP}$ will basically scale the experimental IP signal to the same scale as the experimental input signal, using the spike-in as a link between the two samples.
This makes it natural to subtract the normalized input signal from the normalized IP signal: since they are on the same scale, the resulting coverage can be interpreted as reporting how much more IP signal was detected than was expected based on the input.

\subsection{TSS-seq data analysis}

TSS-seq data analysis was performed as described in section \ref{subsec:tss_seq}, except the experimental genome was \textit{S. pombe} and the spike-in genome was \textit{S. cerevisiae}.

Reannotation of the 5$^\prime$ ends of transcripts was performed as described in section \ref{subsubsec:tss_reannotation}, using transcript and ORF annotations from PomBase \citep{lock2018}, and four replicates of Spt5 non-depleted TSS-seq data.

\subsection{MNase-seq data analysis}

MNase-seq data analysis was performed as described in section \ref{subsec:mnase_seq}, except the \textit{S. pombe} genome was used and no spike-in normalization was performed because no spike-in was included in the experiment.

\newpage
\bibliographystyle{apalike}
\begingroup
    \singlespacing
    \bibliography{references/spt5}
\endgroup
