\chapter{Genomics of transcription elongation factor Spt5}
\label{chapter:five}

\section{Collaborators}

\begin{description}[align=right, leftmargin=!, labelwidth=5cm, noitemsep]
    \item [Ameet Shetty] generated TSS-seq, MNase-seq, NET-seq,\\RNA-seq, and ChIP-seq libraries
\end{description}

\section{Introduction to Spt5 and prior work}

Relevant information about Spt5 is summarized as follows \citep{shetty2017}:

\begin{itemize}[nosep, topsep=.5em]
    \item Spt5 is the only transcription factor known to be conserved in all three domains of life \citep{hartzog2013, werner2012}.
    \item Spt5 co-localizes with elongating RNA Pol II \citep{mayer2010, rahl2010}.
    \item Spt5 binds over the Pol II clamp domain, likely stabilizing the elongation complex \citep{hirtreiter2010, klein2011, martinez-rucobo2011}.
    \item Spt5 physically recruits factors to the elongating transcription complex, in a manner dependent on the modification status of its C-terminal region (CTR) \citep{hartzog2013}:
    \begin{itemize}[nosep]
        \item in its unphosphorylated state, the CTR aids in recruiting the mRNA capping enzyme \citep{doamekpor2014, doamekpor2015, schneider2010, wen1999}
        \item in its phosphorylated state, the CTR recruits the Paf1 complex, which is important for Pol II elongation \citep{liu2009, mbogning2013, wier2013, zhou2009}
        \item Spt5 helps to recruit mRNA 3' end processing factors \citep{mayer2012, stadelmayer2014, yamamoto2014}.
        \item Spt5 helps to recruit the Rpd3S histone deacetylase complex \citep{drouin2010}.
    \end{itemize}
\end{itemize}

\begin{wrapfigure}[11]{r}{3in}
    \begin{tikzpicture}[x=3in, y=1in]
        \draw [line width=2pt] (0,0.1) -- (0.95,0.1);
        \draw [->, line width=3pt] (0.1,0.1) -- (0.1,0.25) -- (0.25,0.25);
        \draw [line width=2pt] (0.175,0.6) -- (0.175, 0.32);
        \draw [line width=2pt] (0.15, 0.32) -- (0.20, 0.32);
        \node at (0.175, 0.7) {\normalsize thiamine};
        \draw [fill=lightpurple, line width=1pt] (0.28, 0) rectangle (0.8, 0.2);
        \node at (0.54, 0.1) {\normalsize spt5};
        \draw [->, line width=2pt] (0.33, 0.3) to [out=90, in=180] (0.5,0.7);
        \draw [fill=lightpurple, line width=1pt] (0.62, 0.7) circle [x radius=0.1, y radius=0.15];
        \node at (0.62, 0.7) {\normalsize Spt5};
        \draw [->, line width=2pt] (0.73, 0.7) to (0.85, 0.7);
        \node at (0.77, 1) {\normalsize auxin};
        \draw [, line width=2pt] (0.77, 0.9) to [out=270, in=180] (0.80,0.7);
        \node at (0.9, 0.7) {\scalebox{2}{$\varnothing$}};
    \end{tikzpicture}
    \caption[Diagram of the dual-shutoff system used to deplete Spt5 from \textit{S. pombe}]{Diagram of the dual-shutoff system used to deplete Spt5 from \textit{S. pombe}. Spt5 is expressed from a thiamine-repressible promoter, and tagged with an auxin-inducible degron tag for specific degradation upon addition of auxin.}
    \label{fig:five_depletion_diagram}
\end{wrapfigure}

\lipsum

\begin{wrapfigure}[14]{r}{3in}
    \includegraphics[width=3in]{figures/five/five_summary_metagenes.pdf}
    \caption[Average Spt5 ChIP-seq, RNAPII ChIP-seq, and sense NET-seq signal over non-overlapping coding genes, from Spt5 depleted and non-depleted cells.]{Average Spt5 ChIP-seq, RNAPII ChIP-seq, and sense NET-seq signal in Spt5 non-depleted and depleted cells, over 1989 non-overlapping coding genes scaled from TSS to CPS. The solid line and shading are the median and inter-quartile range of the mean spike-in normalized coverage over two replicates, in non-overlapping 20 bp bins.}
    \label{fig:five_suummary_metagenes}
\end{wrapfigure}

\lipsum[2]

\begin{wrapfigure}[6]{r}{4.5in}
    \includegraphics[width=4.5in]{figures/five/five_rnapii_phosphomark_enrichment.pdf}
    \caption[Enrichment of RNAPII phospho-serine 5 and phospho-serine 2 over non-overlapping coding genes, in Spt5 depleted and non-depleted cells.]{Caption wsdasdr zzzz.}
    \label{fig:five_rnapii_phosphomark_enrichment}
\end{wrapfigure}

\begin{figure}
    \includegraphics[width=3in]{figures/five/five_rnaseq_heatmaps.pdf}
    \caption[Heatmaps of antisense RNA-seq signal from Spt5 depleted and non-depleted cells, over non-overlapping coding genes.]{Caption wsdasdr zzzz.}
    \label{fig:five_rnaseq_heatmaps}
\end{figure}

\lipsum[2]

\section{TSS-seq results from Spt5 depletion}

\begin{wrapfigure}[6]{r}{3in}
    \includegraphics[width=3in]{figures/five/five_tss_diffexp_summary.pdf}
    \caption[Bar plot of the number of TSS-seq peaks of various genomic classes differentially expressed in Spt5 depleted versus non-depleted cells.]{Caption wsdasdr zzzz.}
    \label{fig:five_tss_diffexp_summary}
\end{wrapfigure}

\lipsum[1]

\begin{figure}
% \includegraphics[width=6in]{figures/stress/stress_promoter_tss_polyenrichment.pdf}
\caption[Heatmaps of antisense TSS-seq, RNA-seq, and NET-seq signal from Spt5 depleted and non-depleted cells, over genes with Spt5-depletion-induced antisense TSSs.]{Caption wsdasdr zzzz.}
% \label{fig:stress_promoter_tss_polyenrichment}
\end{figure}

\section{MNase-seq results from Spt5 depletion}

\begin{wrapfigure}[10]{r}{3in}
    \includegraphics[width=3in]{figures/five/five_mnase_metagene.pdf}
    \caption[Average MNase-seq dyad signal from Spt5 depleted and non-depleted cells, over non-overlapping coding genes.]{Caption wsdasdr zzzz.}
    \label{fig:five_mnase_metagene}
\end{wrapfigure}

\lipsum[1]

\subsection{MNase-seq profile at Spt5-depletion-induced antisense TSSs}

\begin{figure}
% \includegraphics[width=6in]{figures/stress/stress_promoter_tss_polyenrichment.pdf}
\caption[A figure showing MNase-seq signal around Spt5-depletion-induced antisense TSSs.]{Caption wsdasdr zzzz.}
% \label{fig:stress_promoter_tss_polyenrichment}
\end{figure}

\section{Sequence motifs enriched at antisense TSSs}

\begin{figure}
% \includegraphics[width=6in]{figures/stress/stress_promoter_tss_polyenrichment.pdf}
\caption[A figure showing motifs enriched upstream of Spt5-depletion-induced antisense TSSs.]{Caption wsdasdr zzzz.}
% \label{fig:stress_promoter_tss_polyenrichment}
\end{figure}

\section{Discussion}

\section{Methods}

\subsection{A note on spike-in normalization for ChIP-seq experiments with input samples}

In the course of determining how to do spike-in normalization for ChIP-seq libraries, I discovered the following error in a published spike-in normalization method.
Throughout the following explanation, I use `experimental' and `spike-in' to refer to the two genomes present in the experiment, e.g., experimental signal and spike-in signal.

The goal when including spike-ins in a ChIP-seq experiment is to be able to normalize the experimental signal, such that the normalized signal is proportional to the absolute abundance of the factor being immunoprecipitated.
A straightforward method to accomplish this normalization is to linearly scale the experimental signal of a library by a normalization factor, which we will call $\alpha$.
To calculate $\alpha$ for each library, we can use the fact that a normalized `spike-in signal' should be the same for all libraries, since the biological state of the spike-in cells is the same in all libraries.
The key to correctly determining $\alpha$ is defining exactly what this spike-in signal is.

The measurement we begin with for determination of the spike-in signal of a library is the number of reads in the library which map uniquely to the spike-in genome ($R_{\text{spike}}$).
This value will vary based on two factors: the sequencing depth of the library, and the proportion of cells which were spike-in cells ($\phi$):
\begin{align*}
    R_{\text{spike}} &\equiv \text{the number of reads in the library mapping uniquely to the spike-in genome}; \\
    \phi &\equiv \text{the proportion of spike-in cells in the sample}.
\end{align*}
However, the derivation of $\alpha$ is more easily understood in terms of absolute cell numbers rather than $\phi$:
\begin{align*}
    C_{\text{exp}} &\equiv \text{the number of experimental cells used to prepare a library}; \\
    C_{\text{spike}} &\equiv \text{the number of spike-in cells used to prepare a library}. \\
\end{align*}
We can express the \textbf{number of spike-in reads per spike-in cell} by simply taking the fraction $\frac{R_{\text{spike}}}{C_{\text{spike}}}$.
We know that the biological state of a spike-in cell is the same regardless of which sample it belongs to, so we \textit{could} set $\frac{R_{\text{spike}}}{C_{\text{spike}}}$ equal to all samples in order to calculate $\alpha$.
However, this would not account for differences in $\phi$ between samples: Two libraries representing the same condition and sequenced to the same depth should have equivalent values of $\frac{R_{\text{spike}}}{C_{\text{spike}}}$, which does not hold true if they differed in the proportion of spike-in added.

The metric for `spike-in signal' that leads to the correct expression for $\alpha$ is the \textbf{number of spike-in reads per spike-in cell \textit{per experimental cell}}:
\begin{align*}
    & \frac{ \frac{R_\text{spike}}{C_\text{spike}}}{C_\text{exp}} \\
    = & \frac{R_\text{spike} C_\text{exp}}{C_\text{spike}}.
\end{align*}
From here, it's simple to calculate $\alpha$ by setting this value to be equal for all samples.
Since the actual value of the spike-in signal doesn't matter as long as it is equal for all libraries, we can arbitrarily set it to $1$ for convenience:
\begin{align*}
    \alpha \frac{R_\text{spike} C_\text{exp}}{C_\text{spike}} &= 1 \\
    \alpha &= \frac{C_\text{spike}}{R_\text{spike} C_\text{exp}}.
\end{align*}
Notice that only the ratio of spike-in to experimental cells is needed to calculate $\alpha$, and not the absolute number of spike-in and experimental cells.
We can rewrite this expression in terms of $\phi$, the proportion of the sample that was spike-in cells:
\begin{align*}
    \phi &= \frac{C_\text{spike}}{C_\text{spike} + C_\text{exp}} \\
    C_\text{spike} & = \phi \left(C_\text{spike} + C_\text{exp} \right) \\
    C_\text{spike} \left(1-\phi \right) & = \phi C_\text{exp} \\
    \frac{C_\text{spike}}{C_\text{exp}} & = \frac{\phi}{1-\phi} & \alpha &= \frac{C_\text{spike}}{R_\text{spike} C_\text{exp}} \\
                                        && \alpha &= \frac{\phi}{R_\text{spike} \left(1-\phi \right)}.
\end{align*}
This form for $\alpha$ differs from the one presented in \citet{orlando} with no derivation:
\begin{align*}
    \alpha &= \frac{\phi}{R_\text{spike} \left(1-\phi \right)} & \alpha_\text{orlando} &= \frac{\phi}{R_\text{spike}}.
\end{align*}
Working through a few examples with both versions of $\alpha$ will reveal that $\alpha_\text{orlando}$ leads to incorrect normalization when $\phi$ is not equivalent for all samples.

In the first example, we will vary sequencing depth between two libraries, keeping everything else constant.
Consider a single ChIP library prep in which 20\% of the cells were spike-in cells (i.e., $\phi=0.2$).
The library is then unevenly split into two aliquots and sequenced.
One library has four times the reads of the other library.
\begin{align*}
    R_{\text{spike}_1} &= 1 & R_{\text{spike}_2} &= 4 \\
    R_{\text{exp}_1} &= 4 & R_{\text{exp}_2} &= 16 \\
\end{align*}
\begin{align*}
    \alpha_1 &= \frac{\phi}{R_{\text{spike}_1} \left(1-\phi \right)} &
    \alpha_2 &= \frac{\phi}{R_{\text{spike}_2} \left(1-\phi \right)} &
    \alpha_{\text{orlando}_1} &= \frac{\phi}{R_{\text{spike}_1}} &
    \alpha_{\text{orlando}_2} &= \frac{\phi}{R_{\text{spike}_2}} \\
    \alpha_1 &= \frac{0.2}{1 \left(0.8 \right)} &
    \alpha_2 &= \frac{0.2}{4 \left(0.8 \right)} &
    \alpha_{\text{orlando}_1} &= \frac{0.2}{1} &
    \alpha_{\text{orlando}_2} &= \frac{0.2}{4} \\
    \alpha_1 &= \frac{4}{16} &
    \alpha_2 &= \frac{1}{16} &
    \alpha_{\text{orlando}_1} &= \frac{4}{20} &
    \alpha_{\text{orlando}_2} &= \frac{1}{20}.
\end{align*}
The total levels of spike-in normalized experimental signal can be found for each library by multiplying $\alpha$ by $R_\text{exp}$, for our version of $\alpha$,
\begin{align*}
    \text{signal}_1 &= \alpha_1 R_{\text{exp}_1}  &
    \text{signal}_2 &= \alpha_2 R_{\text{exp}_2}  \\
    \text{signal}_1 &=  \frac{4}{16} \left(4 \right)  &
    \text{signal}_2 &=  \frac{1}{16} \left(16 \right)  \\
    \text{signal}_1 &=  1 &
    \text{signal}_2 &=  1 \\
\end{align*}
and for $\alpha_\text{orlando}$:
\begin{align*}
    \text{signal}_{\text{orlando}_1} &= \alpha_{\text{orlando}_1} R_{\text{exp}_1} &
    \text{signal}_{\text{orlando}_2} &= \alpha_{\text{orlando}_2} R_{\text{exp}_2} \\
    \text{signal}_{\text{orlando}_1} &= \frac{4}{20} \left(4\right) &
    \text{signal}_{\text{orlando}_2} &= \frac{1}{20} \left(16\right) \\
    \text{signal}_{\text{orlando}_1} &= 0.8 &
    \text{signal}_{\text{orlando}_2} &= 0.8 \\
\end{align*}
Only the relative abundances within normalization methods matter, so in this case both calculations correctly normalized for library size and say that the normalized signal in the two libraries are the same.

Now let's consider two libraries from two different conditions with $\phi=0.1$.
In condition 2, there is a known global decrease in experimental signal expected.
This time, we will skip the algebra:
\begin{align*}
    R_{\text{spike}_1} &= 1 & R_{\text{spike}_2} &= 4 \\
    R_{\text{exp}_1} &= 9 & R_{\text{exp}_2} &= 6 \\
\end{align*}
\begin{align*}
    % \alpha_1 &= \frac{\phi}{R_{\text{spike}_1} \left(1-\phi \right)} &
    % \alpha_2 &= \frac{\phi}{R_{\text{spike}_2} \left(1-\phi \right)} &
    % \alpha_{\text{orlando}_1} &= \frac{\phi}{R_{\text{spike}_1}} &
    % \alpha_{\text{orlando}_2} &= \frac{\phi}{R_{\text{spike}_2}} \\
    % \alpha_1 &= \frac{0.1}{1 \left(0.9 \right)} &
    % \alpha_2 &= \frac{0.1}{4 \left(0.9 \right)} &
    % \alpha_{\text{orlando}_1} &= \frac{0.1}{1} &
    % \alpha_{\text{orlando}_2} &= \frac{0.1}{4} \\
    \alpha_1 &= \frac{4}{36} &
    \alpha_2 &= \frac{1}{36} &
    \alpha_{\text{orlando}_1} &= \frac{4}{40} &
    \alpha_{\text{orlando}_2} &= \frac{1}{40}
\end{align*}
\begin{align*}
    % \text{signal}_1 &= \alpha_1 R_{\text{exp}_1}  &
    % \text{signal}_2 &= \alpha_2 R_{\text{exp}_2}  &
    % \text{signal}_{\text{orlando}_1} &= \alpha_{\text{orlando}_1} R_{\text{exp}_1} &
    % \text{signal}_{\text{orlando}_2} &= \alpha_{\text{orlando}_2} R_{\text{exp}_2} & \\
    % \text{signal}_1 &=  \frac{4}{36} \left(9 \right)  &
    % \text{signal}_2 &=  \frac{1}{36} \left(6 \right)  &
    % \text{signal}_{\text{orlando}_1} &= \frac{4}{40} \left(9\right) &
    % \text{signal}_{\text{orlando}_1} &= \frac{1}{40} \left(6\right) & \\
    \text{signal}_1 &=  1 &
    \text{signal}_2 &=  1/6 &
    \text{signal}_{\text{orlando}_1} &= 0.9 &
    \text{signal}_{\text{orlando}_2} &= 0.15 &
\end{align*}

Both methods correctly detect that experimental signal levels in library 2 are 1/6th that of library 1.

Finally, let's consider two libraries from the same condition which were spiked in with different amounts of spike-in cells. Both libraries are sequenced to the same depth. Since the libraries are from the same condition, we expect their total experimental signal to be the same after normalization, even though they had different amounts of spike-in added.
\begin{align*}
    \phi_1 &= 0.2 & \phi_2 &=0.4 \\
    R_{\text{spike}_1} &= 2 & R_{\text{spike}_2} &= 4 \\
    R_{\text{exp}_1} &= 8 & R_{\text{exp}_2} &= 6 \\
\end{align*}
\begin{align*}
    \alpha_1 &= \frac{\phi_1}{R_{\text{spike}_1} \left(1-\phi_1 \right)} &
    \alpha_2 &= \frac{\phi_2}{R_{\text{spike}_2} \left(1-\phi_2 \right)} &
    \alpha_{\text{orlando}_1} &= \frac{\phi_1}{R_{\text{spike}_1}} &
    \alpha_{\text{orlando}_2} &= \frac{\phi_2}{R_{\text{spike}_2}} \\
    \alpha_1 &= \frac{0.2}{2 \left(0.8 \right)} &
    \alpha_2 &= \frac{0.4}{4 \left(0.6 \right)} &
    \alpha_{\text{orlando}_1} &= \frac{0.2}{2} &
    \alpha_{\text{orlando}_2} &= \frac{0.4}{4} \\
    \alpha_1 &= \frac{3}{24} &
    \alpha_2 &= \frac{4}{24} &
    \alpha_{\text{orlando}_1} &= \frac{1}{10} &
    \alpha_{\text{orlando}_2} &= \frac{1}{10}
\end{align*}
\begin{align*}
    \text{signal}_1 &= \alpha_1 R_{\text{exp}_1}  &
    \text{signal}_2 &= \alpha_2 R_{\text{exp}_2}  \\
    \text{signal}_1 &=  \frac{3}{24} \left(8 \right)  &
    \text{signal}_2 &=  \frac{4}{24} \left(6 \right)  \\
    \text{signal}_1 &=  1 &
    \text{signal}_2 &=  1 \\
\end{align*}
\begin{align*}
    \text{signal}_{\text{orlando}_1} &= \alpha_{\text{orlando}_1} R_{\text{exp}_1} &
    \text{signal}_{\text{orlando}_2} &= \alpha_{\text{orlando}_2} R_{\text{exp}_2} & \\
    \text{signal}_{\text{orlando}_1} &= \frac{1}{10} \left(8\right) &
    \text{signal}_{\text{orlando}_2} &= \frac{1}{10} \left(6\right) & \\
    \text{signal}_{\text{orlando}_1} &= 0.8 &
    \text{signal}_{\text{orlando}_2} &= 0.6 \\
\end{align*}
Here, our method correctly normalizes the two samples to the same total experimental signal while using the Orlando $\alpha$ results in an apparent decrease in signal in library 2.
This is because the Orlando $\alpha$ fails to account for the fact that when you add more spike-in to a sample, you necessarily decrease the proportion of the sample that is experimental.
In most experiments with spike-ins, this isn't really a problem because we assume that $\phi$ is the same for all samples.
However, with ChIP-seq experiments that include input samples, if we assume that the experimental and spike-in input sample read counts are proportional to the amounts of experimental and spike-in cells mixed, we can plug these values in for values of $\phi$ to get a more reliable estimation of experimental signal levels.
In this case, it becomes important to use the correct equation for $\alpha$.

So, putting everything together, here's how I use the spike-in to normalize an IP ChIP-seq library paired with an input ChIP-seq library.

As stated above, we assume that the experimental and spike-in read counts in the input sample are proportional to the numbers of experimental and spike-in cells used to prepare the library:
\begin{align*}
    R_{\text{input}_\text{exp}} \propto C_\text{exp}, \\
    R_{\text{input}_\text{spike}} \propto C_\text{spike}
\end{align*}
Therefore, we can plug these values in for $C$ for both the input and IP libraries (using the form of $\alpha$ without $\phi$):
\begin{align*}
    \alpha_\text{input} &= \frac{C_{\text{input}_\text{spike}}}{R_{\text{input}_\text{spike}} C_{\text{input}_\text{exp}}} &
    \alpha_\text{IP} &= \frac{C_{\text{input}_\text{spike}}}{R_{\text{IP}_\text{spike}} C_{\text{input}_\text{exp}}} \\
    \alpha_\text{input} &\propto \frac{R_{\text{input}_\text{spike}}}{R_{\text{input}_\text{spike}} R_{\text{input}_\text{exp}}} &
    \alpha_\text{IP} &\propto \frac{R_{\text{input}_\text{spike}}}{R_{\text{IP}_\text{spike}} R_{\text{input}_\text{exp}}} \\
    \alpha_\text{input} &\propto \frac{1}{R_{\text{input}_\text{exp}}} &
\end{align*}
Notice how $\alpha_\text{input}$ reduces down to normalizing by the experimental library size, with no dependence at all on the spike-in.
This makes sense because the input always represents the same state, regardless of how much spike-in is added to it.
The function of the spike-in in the input is only to allow us to estimate abundances in the corresponding IP library.
Rewriting $\alpha_\text{IP}$ in the form 
\begin{align*}
    \alpha_\text{IP} &\propto \frac{1}{R_{\text{IP}_\text{spike}} \frac{R_{\text{input}_\text{exp}}}{R_{\text{input}_\text{spike}}}}
\end{align*}
shows that $\alpha_\text{IP}$ will basically scale the experimental IP signal to the same scale as the experimental input signal, using the spike-in as a link between the two samples.
This makes it natural to subtract the normalized input signal from the normalized IP signal: since they are on the same scale, the resulting coverage can be interpreted as reporting how much more IP signal was detected than was expected based on the input.

\newpage
\bibliographystyle{apalike}
\begingroup
    \singlespacing
    \bibliography{references/spt5}
\endgroup
