\chapter{Introduction}

\section{A brief introduction to transcription}

In eukaryotic cells, transcription of protein-coding genes is carried out by the protein complex RNA polymerase II (Pol II), and broadly occurs in three sequential stages of transcription initiation, elongation, and termination \citep{shandilya2012}.
During each of these stages, the Pol II complex is associated with distinct sets of factors which modulate the activity of Pol II and carry out co-transcriptional processes such as RNA capping, RNA splicing, histone modification, RNA cleavage, and RNA polyadenylation.
Given how fundamental transcription is to gene expression, it is unsurprising that every stage of transcription is highly regulated.

To get a rough idea of just how tightly transcription is regulated, it is useful to consider a back-of-the-envelope calculation of the specificity of transcription initiation in the human genome.
That is, what proportion of the human genome at which transcription could initiate does transcription initiation actually occur?

The number of positions at which transcription could theoretically initiate is simply the size of the genome: The human genome is approximately three billion base pairs in length (BNID 111378, \citet{griffin2009}), and since each base pair can be transcribed from each of its two strands, there are $6 \times 10^9$ available positions.

The number of positions at which transcription \textit{does} initiate can be estimated from the number of genes transcribed by Pol II and the number of positions that Pol II initiates from for each gene.
At last count, the human genome contains about twenty thousand protein-coding genes \citep{encode2012}.
To be conservative in our estimate with regards to specificity, we will assume that all twenty thousand genes are expressed.
We also know that protein-coding genes are only a subset of the genes transcribed by Pol II: Pol II also transcribes multiple classes of non-coding genes, including enhancers and long non-coding RNAs \citep{kaikkonen2018}.
Compared to protein-coding genes, the number of non-coding genes is less certain.
If we assume that there are five non-coding genes for each coding gene, this brings our estimate of the number of genes transcribed by Pol II to $1.2 \times 10^5$ genes.

As you will see from yeast transcription start site data in later chapters, transcription initiation for a single gene generally occurs at multiple nucleotides, generating multiple major transcript isoforms per gene.
Assuming that there are, on average, five major transcription start sites (TSSs) per gene, the proportion of the human genome at which transcription initiation occurs is
\begin{align*}
    \frac{\left(1.2 \times 10^5 \; \text{genes}\right) \left(5 \; \frac{\text{TSSs}}{\text{gene}} \right)}
         {\left(6 \times 10^9 \; \text{possible TSSs} \right)}
    &= 1 \times 10^{-4}.
\end{align*}
Our rough estimate says that, when presented with ten thousand positions to choose from, RNA polymerase starts transcription from only one!\footnote{A similar conclusion is reached by examining ENCODE CAGE-seq data: At the time of writing, ENCODE reports roughly 150,000 TSS peaks across 30 cell types/cell lines. Assuming the signal is concentrated at 5 nucleotides per peak, then $\frac{\left(1.5 \times 10^5 \text{peaks} \right) \left(5 \frac{\text{nt}}{\text{peak}} \right)}{6 \times 10^9 \text{nt}} = \frac{1}{8000}$.}

Many factors are known to contribute to this remarkable specificity.
Most notably, transcription initiation requires the presence of specific DNA sequence motifs, which increase the probability of Pol II binding to DNA together with necessary initiation factors \citep{haberle2018}.
That factors known to associate with Pol II during transcription initiation control transcription initiation is unsurprising.
A less obvious fact is that some transcription \textit{elongation} factors, including histone chaperones and histone modification enzymes, also play a role in determining where transcription initiation is allowed to occur \citep{kaplan2003, cheung2008, hennig2013}.
Evidence suggests that these elongation factors are likely required to maintain normal chromatin structure over transcribed regions, and that the disruption of normal chromatin structure allows Pol II to initiate transcription in regions which are normally inacessible.
Chapters \ref{chapter:six} and \ref{chapter:five} of this dissertation describe our studies of \textbf{Spt6} and \textbf{Spt5}, two of the transcription elongation factors involved in this process.
One phenotype observed when these factors are disrupted is \textbf{intragenic transcription}, transcription appearing to arise from within protein-coding sequences.
In chapter \ref{chapter:stress}, I describe our efforts to understand how intragenic transcription might play a role in the cellular response to various stress conditions.
The remainder of this introduction provides a brief overview of the considerations taken into account in order to make the data analyses behind this dissertation as transparent and reproducible as possible.

\section{Reproducible data analysis for genomics}

My role in the projects in this dissertation is a mix of \href{https://blog.insightdatascience.com/data-science-vs-data-engineering-62da7678adaa}{\textbf{data scientist}} and \href{https://blog.insightdatascience.com/data-science-vs-data-engineering-62da7678adaa}{\textbf{data engineer}}: I build pipelines for processing (usually genomic) datasets, taking raw data through processing, statistical analysis, and data visualization.
This mostly entails surveying available tools, selecting the tools most suitable for the task, and coding solutions to problems when existing tools are insufficient.

The analysis of complex datasets like those in genomics presents challenges to achieving transparency and reproducibility when reporting methods and results.
In building the data analysis pipelines behind the results of this dissertation, I have tried to meet these challenges by following best practices that would be standards for publication in an ideal world.
All of my data analyses are open source (\href{https://github.com/winston-lab}{github.com/winston-lab}), and are designed to be reproducible by others: For all publications, a self-contained archive is uploaded which includes everything needed to go from raw data to the figures and results of the publication (e.g.,\\\url{https://doi.org/10.5281/zenodo.1409826}).
This level of accessibility is greatly facilitated by building data analyses using Snakemake \citep{koster2012}, one of several available frameworks for workflow management \citep{voss2017, ditommaso2017}.
Snakemake's scalable execution and its ability to specify dependencies in virtual environments allow workflows to truly be reproducible: data analyses can be re-run on personal computers, computing clusters, or cloud environments, and the exact versions of the software used when initially running the data analysis will automagically be deployed.

Open sharing of data and code like this is essential to the scientific process.
When analysis pipelines routinely consist of tens of steps with tens of parameters each, seeing the data and code is the only way for those interested to know exactly how the data were handled.
Altogether, this allows for more informed evaluation of results from the literature, as well as the possibility of finding and correcting errors in analysis.

\clearpage
\bibliographystyle{apalike}
\begingroup
    \singlespacing
    \bibliography{references/introduction,references/spt6,references/spt5}
\endgroup

