\chapter{Genomics of transcription elongation factor Spt6}

\section{Collaborators}

\begin{description}[align=right, labelwidth=5cm, noitemsep]
    \item [Steve Doris] optimized TSS-seq and ChIP-nexus protocols
    \item [] generated TSS-seq and ChIP-nexus libraries
    \item [Olga Viktorovskaya] generated MNase-seq libraries
    \item [Magdalena Murawska] generated NET-seq libraries
    \item [Dan Spatt] various experiments for publication
\end{description}

\section{Introduction to Spt6 and intragenic transcription}

% The work described in this chapter relates to understanding how a eukaryotic cell specifies which sites in its genome are permitted to become sites of transcription initiation.
% To get a rough idea of the specificity of transcription initiation, it is useful to start with a simple back-of-the-envelope calculation of the proportion of the human genome at which transcription initiation occurs.
% The human genome is approximately three billion base pairs in length, and each base pair can potentially be transcribed from each of its two strands.
% Each gene in the genome can have multiple transcription start sites (TSSs), which I assume to be five in number for the average gene.
% At last count, the human genome contains about twenty thousand protein-coding genes.
% To be conservative in our estimate with regards to specificity, we assume that all twenty thousand genes are expressed, which leads to the following proportion:
% \begin{align*}
%     \frac{\left(2 \times 10^4 \; \text{genes}\right) \left(5 \; \frac{\text{TSS}}{\text{gene}} \right)}
%          {\left(3 \times 10^9 \; \text{base pairs} \right) \left(2 \; \frac{\text{TSS}}{\text{base pair}} \right)}.
% \end{align*}
% However, this expression underestimates the extent of transcription initiation by only considering protein-coding genes, neglecting the many classes of noncoding genes present in the genome.
% If we assume that there are five noncoding genes for each coding gene, the updated expression,
% \begin{align*}
%     \frac{\left(1.2 \times 10^5 \; \text{genes}\right) \left(5 \; \frac{\text{TSS}}{\text{gene}} \right)}
%          {\left(3 \times 10^9 \; \text{base pairs} \right) \left(2 \; \frac{\text{TSS}}{\text{base pair}} \right)}
%     &= 0.0001,
% \end{align*}
% says that when presented with a thousand positions to choose from, RNA polymerase chooses just one to start transcribing from!

% Where transcription initiates is determined in large part by DNA sequence: the presence of certain sequence motifs increases the probability that RNA polymerase binds to DNA along with numerous co-factors required for initiation.
% However, DNA sequence alone does not entirely account for the specificity of transcription initiation.
% Genetic studies in yeast first showed that some transcription \textit{elongation} factors, including histone chaperones and histone modification enzymes, play a role in restricting where transcription is allowed to initiate \citep{kaplan2003, cheung2008, hennig2013}.
% In this project, we study the role of a particular transcription elongation factor called \textbf{Spt6} in this process.
% Many years of research on Spt6 is summarised as follows \citep{doris2018}:

% \begin{itemize}[nosep, topsep=.5em]
% \item Spt6 interacts directly with:
% 	\begin{itemize}[nosep]
% 	\item RNA polymerase II (RNAPII) \citep{close2011, diebold2011, liu2011, sdano2017, sun2010, yoh2007}
% 	\item histones \citep{bortvin1996, mccullough2015}
% 	\item the essential factor Spn1 (IWS1) \citep{diebold2010b, li2018, mcdonald2010}
% 	\end{itemize}
% \item Spt6 is believed to function primarily as an elongation factor based on:
% 	\begin{itemize}[nosep]
% 	\item association with elongating RNAPII \citep{andrulis2000, ivanovska2011, kaplan2000, mayer2010}
% 	\item ability to enhance elongation in vitro \citep{endoh2004} and in vivo \citep{ardehali2009}
% 	\end{itemize}
% \item Spt6 has been shown to regulate initiation in some cases \citep{adkins2006, ivanovska2011}
% \item Spt6 regulates:
% 	\begin{itemize}[nosep]
% 	\item chromatin structure \citep{bortvin1996, degennaro2013, ivanovska2011, jeronimo2015, kaplan2003, perales2013, vanbakel2013}
% 	\item histone modifications, including:
% 		\begin{itemize}[nosep]
% 		\item H3K36 methylation \citep{carrozza2005, chu2006, yoh2008, youdell2008}
% 		\item in some organisms, H3K4 and H3K27 methylation \citep{begum2012, chen2012, degennaro2013, wang2017, wang2013}
% 		\end{itemize}
% 	\end{itemize}
% \item Spt6 is likely a histone chaperone required to reassemble nucleosomes in the wake of transcription \citep{duina2011}.
% \end{itemize}

\begin{wrapfigure}[18]{r}{3in}
    \centering
    \includegraphics[width=3in]{figures/six/six_spt6_western.pdf}
    \caption[Western blot showing Spt6 protein levels in wild-type and \textit{spt6-1004} cells, at 30\textdegree C and after 80 minutes at 37 \textdegree C.]{Western blot showing Spt6 protein levels in wild-type and \textit{spt6-1004} cells, at 30\textdegree C and after 80 minutes at 37 \textdegree C. Immunoblotting was performed using $\alpha$-FLAG antibody to detect Spt6 and $\alpha$-Myc antibody to detect Dst1 from a spike-in strain. The quantification shown is the mean $\pm$ standard deviation of three blots.}
    \label{fig:six_spt6_western}

    \vspace{0.5em}

    \centering
    \includegraphics[width=3in]{figures/six/six_gene_diagram.pdf}
    \caption[Diagram of transcript classes.]{Diagram of transcript orientation with respect to coding DNA sequences, for the categories of transcripts referred to in this document.}
    \label{fig:six_gene_diagram}
\end{wrapfigure}

Studies in the yeasts \textit{Saccharomyces cerevisiae} and \textit{Schizosaccharomyces pombe} have previously examined the requirement for Spt6 in normal transcription \citep{cheung2008, degennaro2013, kaplan2003, pathak2018, uwimana2017, vanbakel2013}.
As Spt6 is essential for viability in \textit{S. cerevisiae}, many of these studies use the same temperature-sensitive \textit{spt6} mutant used in this project, \textbf{\textit{spt6-1004}}, which encodes an in-frame deletion of a helix-hairpin-helix domain within Spt6 \citep{kaplan2003}.
When \textit{spt6-1004} cells are shifted from 30\textdegree C to 37\textdegree C for 80 minutes, bulk Spt6 protein levels are depleted to about 20\% of wild-type levels (Figure \ref{fig:six_spt6_western}).
The most notable phenotype of the \textit{spt6-1004} mutant is the appearance of \textbf{intragenic transcripts}, transcripts which appear to arise from within protein-coding sequences, in both sense and antisense orientations relative to the coding gene (Figure \ref{fig:six_gene_diagram}) \citep{cheung2008, degennaro2013, kaplan2003, uwimana2017}.

\begin{SCfigure}[40][h]
    \centering
    \includegraphics[width=4.4in]{figures/six/six_aat_assay_comparison.pdf}
    \caption[RNA-seq, TSS-seq, and TFIIB ChIP-nexus signal at the \textit{AAT2} gene, in \textit{spt6-1004} after 80 minutes at 37\textdegree C.]{Sense strand RNA-seq signal, sense strand TSS-seq signal, and TFIIB ChIP-nexus protection at the \textit{AAT2} gene, in \textit{spt6-1004} after 80 minutes at 37\textdegree C.}
    \label{fig:six_aat_assay_comparison}
\end{SCfigure}

Previous genome-wide measurements of transcript levels in \textit{spt6-1004} relied on tiled microarrays \citep{cheung2008} and RNA sequencing \citep{uwimana2017}.
Studying intragenic transcription is difficult with these methods, since the signal for an intragenic transcript in the same orientation as the gene it overlaps is convoluted with the signal from the full-length `genic' transcript (Figure \ref{fig:six_aat_assay_comparison}) \citep{cheung2008, lickwar2009}. Therefore, these methods can only discover intragenic transcripts which are highly expressed relative to the corresponding genic transcript, and are likely to find many false positives.
Additionally, these methods are assays of steady-state RNA levels, which makes them unable to distinguish whether the intragenic transcripts observed in \textit{spt6-1004} result from: A) new intragenic transcription initiation in the mutant, B) reduced decay of intragenic transcripts which are rapidly degraded in wild-type, or C) processing of full-length protein-coding RNAs.

To address these challenges to studying intragenic transcription, we applied two genomic assays to \textit{spt6-1004}: transcription start-site sequencing (\textbf{TSS-seq}), and \textbf{ChIP-nexus of TFIIB}, a component of the RNA polymerase II pre-initiation complex (PIC).
TSS-seq sequences the 5$^\prime$ end of capped and polyadenylated RNAs \citep{arribere2013, malabat2015}, allowing separation of intragenic from genic RNA signals and identification of intragenic transcript starts with single-nucleotide resolution (Figure \ref{fig:six_aat_assay_comparison}).
ChIP-nexus is a high-resolution chromatin immunoprecipitation technique, in which the immunoprecipitated DNA is exonuclease digested up to the bases crosslinked with the protein of interest before sequencing \citep{he2015}.
When applied to the PIC component TFIIB, ChIP-nexus reports where transcription initiation is occurring, thus allowing us to determine if intragenic transcripts in \textit{spt6-1004} result from new transcription initiation.

\section{Data analysis pipelines for TSS-seq and ChIP-nexus}

% In order to use TSS-seq and ChIP-nexus to answer questions about Spt6 and intragenic transcription, I developed analysis pipelines for TSS-seq and ChIP-nexus data.
% The pipelines are written using the Python-based Snakemake workflow specification language \citep{koster2012}, and perform steps including read cleaning \citep{martin2011}, various quality controls \citep{andrews2012}, read alignment \citep{kim2013, langmead2012}, data normalization, coverage track generation \citep{quinlan2010}, peak calling \citep{zhang2008}, differential expression/binding analyses \citep{love2014}, data visualization with clustering, motif enrichment analyses \citep{bailey2015}, and gene ontology analyses \citep{young2010}.
% The Snakemake framework allows these data analyses to be reproducible and scalable from workstations up to computing clusters.
% Updated versions of these pipelines with more details on their capabilities are available at \href{https://github.com/winston-lab}{github.com/winston-lab}.
% In the following subsections I will describe the thought behind only a few of the more novel pipeline steps before moving on to results relating to Spt6 and intragenic transcription.

\subsection{TSS-seq peak calling}

% TSS-seq data from a single region of transcription initiation tends to occur as a cluster of signal distributed over a range of positions, rather than a single nucleotide (Figure \ref{fig:tss_coverage}) \citep{arribere2013, malabat2015}.
% It is reasonable to consider such a cluster of TSS-seq signal as a single entity, because the signals within the cluster are usually highly correlated to one another across different conditions.
% Therefore, to identify TSSs from TSS-seq data and quantify them for downstream analyses such as differential expression, it is necessary to annotate these groups of signal by using the data to perform peak-calling.

% In its current state, the TSS-seq pipeline calls peaks using 1-D \href{https://en.wikipedia.org/wiki/Watershed_(image_processing)}{watershed segmentation}, followed by filtering for reproducibility by the Irreproducible Discovery Rate (IDR) method \citep{li2011}.
% First, a smoothed version of the TSS-seq coverage is generated for each sample using a discretized Gaussian kernel.
% Next, an initial set of peaks is generated by: 1) assigning all nonzero signal in the original, unsmoothed coverage to the nearest local maximum of the smoothed coverage in the direction of positive derivative, and 2) taking the minimum and maximum genomic coordinates of the original coverage assigned to each local maximum as the peak boundaries.
% The peaks are then trimmed to the smallest genomic window that includes 95\% of the original coverage, and the probability of the peak being generated by noise is estimated by a Poisson model where $\lambda$, the expected coverage, is the maximum of the expected coverage over the chromosome and the expected coverage in a window upstream of the peak (as for the ChIP-seq peak caller MACS2 \citep{zhang2008}).
% The influence of local read density on $\lambda$ is intended to reduce false positive peaks within gene bodies, especially for highly expressed genes: Since there are more fragments of RNA present for highly expressed genes, more fragments within the gene body will make it into the final library, even if they are not true 5' ends.
% To generate the final set of peaks, the peaks are ranked by significance under the Poisson model, and filtered by IDR.
% In brief, IDR attempts to separate true peaks from experimental noise based on the intuition that, when peaks in each replicate are independently ranked by a metric such as significance, true peaks will have more similar ranks between replicates than peaks representing noise \citep{li2011}.

% The IDR algorithm currently only works for two replicates.
% Future improvements could include expanding the IDR implementation to handle more replicates and improve the accuracy of peak calling with more data.

\subsection{A note on ChIP-nexus peak calling}

% A number of tools have been created specifically for peak-calling using data from high-resolution ChIP techniques such as ChIP-nexus and ChIP-exo \citep{wang2014, hansen2016}.
% When applied to our TFIIB ChIP-nexus data, these tools tended to split what appeared to be a single TFIIB binding event into multiple peaks.
% This may be because TFIIB has been observed to crosslink to DNA at multiple points (Figure \ref{fig:tfiib_tata}) \citep{rhee2012}, which suggests that while these tools may work well for factors that bind symmetrically with a single crosslinking point on either side, there is still room for improvement when it comes to factors with more complex binding patterns.
% For the purposes of this project, the standard ChIP-seq peak caller MACS2 was used \citep{zhang2008}.

% ChIP-seq peaks lack strand information, as DNA binding factors usually do not bind DNA in a strand-specific manner.
% Because of this, we could not separate intragenic TFIIB peaks into peaks associated with sense or antisense transcription.
% The distinctive shape of the aggregate TFIIB ChIP-nexus signal (Figure \ref{fig:tfiib_tata}) suggests that information about the strand of transcription may be present in the ChIP-nexus binding profile.
% Future work could include learning the direction of transcription from labeled ChIP-nexus training data.

\section{TSS-seq and TFIIB ChIP-nexus results for \textit{spt6-1004}}

To study the relationship between Spt6 and transcription, TSS-seq and TFIIB ChIP-nexus libraries were prepared from wild-type and \textit{spt6-1004} cells, both shifted from 30\textdegree C to 37\textdegree C for 80 minutes.
In wild-type cells, TSS-seq and TFIIB ChIP-nexus recapitulate their expected distributions over the genome: Most TSS signal is restricted to annotated genic TSSs, while most TFIIB signal is localized just upstream of the TSS (Figures \ref{fig:six_tss_seq_heatmaps}, \ref{fig:six_tfiib_heatmap}).
In \textit{spt6-1004}, the signal for both assays infiltrates gene bodies, already suggesting that new transcription initiation does contribute to the intragenic transcription phenotype.
Notably, sense strand TSS-seq signal in \textit{spt6-1004} tends to occur towards the 3$^\prime$ end of genes, while antisense strand TSS-seq signal tends to occur towards the 5$^\prime$ end of genes.

\begin{figure}[H]
\centering
\includegraphics[width=6in]{figures/six/six_tss_seq_heatmaps.pdf}
\caption[Heatmaps of sense and antisense TSS-seq signal from wild-type and \textit{spt6-1004} cells, over non-overlapping coding genes.]{Heatmaps of sense and antisense TSS-seq signal from wild-type and \textit{spt6-1004} cells, over 3522 non-overlapping genes aligned by wild-type genic TSS and sorted by annotated transcript length. Data are shown for each gene up to 300 nucleotides 3$^\prime$ of the cleavage and polyadenylation site (CPS), indicated by the white dotted line. Values are the mean of spike-in normalized coverage in non-overlapping 20 nucleotide bins, averaged over two replicates. Values above the 92nd percentile are set to the 92nd percentile for visualization.}
\label{fig:six_tss_seq_heatmaps}
\end{figure}

\begin{wrapfigure}[19]{R}{3in}
    \centering
    \includegraphics[width=3in]{figures/six/six_tfiib_heatmap.pdf}
    \caption[Heatmaps of TFIIB ChIP-nexus protection from wild-type and \textit{spt6-1004} cells, over non-overlapping coding genes]{Heatmaps of TFIIB binding measured by ChIP-nexus, over the same regions shown in Figure \ref{fig:six_tss_seq_heatmaps}. Values are the mean of library-size normalized coverage in non-overlapping 20 bp bins, averaged over two replicates. Values above the 85th percentile are set to the 85th percentile for visualization.}
    \label{fig:six_tfiib_heatmap}
\end{wrapfigure}

The TSS-seq data were quantified by peak calling and differential expression analysis, and classified into genomic categories based on their position relative to coding genes.
As suggested by the heatmap visualization (Figure \ref{fig:six_tss_seq_heatmaps}), we detect significant induction of over 4000 intragenic and antisense TSSs in \textit{spt6-1004} (Figure \ref{fig:six_tss_diffexp_summary}).
Compared to previous studies identifying \textit{spt6-1004} intragenic transcription by tiled microarray and RNA-seq, we identify intragenic transcription at over 1000 additional genes (Figure \ref{fig:six_intragenic_genes_bvenn}) and have the exact start sites of all identified TSSs.
The TSS-seq data also revealed an unexpected downregulation of most genic TSSs: In this experiment, we detected a significant downregulation to levels below 67\% of wild-type levels at 75\% (3579/4792) of genic TSSs (Figure \ref{fig:six_tss_diffexp_summary}).
As a result of intragenic/antisense induction and genic repression, expression levels in \textit{spt6-1004} of all classes of transcripts become similar to one another (Figure \ref{fig:six_tss_expression_levels}).

\begin{figure}[H]
    \centering
    \begin{minipage}[t]{2.875in}
        \centering
        \includegraphics[width=2.875in]{figures/six/six_tss_diffexp_summary.pdf}
        \caption[Bar plot of the number of TSS-seq peaks of various genomic classes differentially expressed in \textit{spt6-1004} versus wild-type.]{Bar plots of the number of TSS-seq peaks differentially expressed in \textit{spt6-1004} after 80 minutes at 37\textdegree C versus wild-type after 80 minutes at 37\textdegree C. The height of each bar is proportional to the total number of peaks in the category, including those not found to be significantly differentially expressed.}
        \label{fig:six_tss_diffexp_summary}
    \end{minipage}\hfill
    \begin{minipage}[t]{2.875in}
        \centering
        \includegraphics[width=2.875in]{figures/six/six_intragenic_genes_bvenn.pdf}
        \caption[Set diagram of the number of genes with \textit{spt6-1004}-induced intragenic transcripts reported in \citet{cheung2008}, \citet{uwimana2017}, and our TSS-seq data.]{Set diagram of the number of genes reported to have \textit{spt6-1004}-induced intragenic transcripts using tiled arrays \citet{cheung2008}, RNA-seq \citet{uwimana2017}, and TSS-seq (this work).}
        \label{fig:six_intragenic_genes_bvenn}
    \end{minipage}
\end{figure}

\begin{wrapfigure}[10]{r}{3in}
    \centering
    \includegraphics[width=3in]{figures/six/six_tss_expression_levels.pdf}
    \caption[Violin plots of expression level distributions for genomic classes of TSS-seq peaks in wild-type and \textit{spt6-1004} cells.]{Violin plots of expression level distributions for genomic classes of TSS-seq peaks in wild-type and \textit{spt6-1004}, both after 80 minutes at 37\textdegree C. Normalized counts are the mean of spike-in size factor normalized counts from two replicates.}
    \label{fig:six_tss_expression_levels}
\end{wrapfigure}

The changes in transcript levels in \textit{spt6-1004} observed by TSS-seq correspond with substantial differences in the pattern of TFIIB binding on the genome.
In contrast to the discrete peaks in promoter regions seen in wild-type, TFIIB in \textit{spt6-1004} binds much more promiscuously, with many loci having TFIIB signal spread over broad regions of the genome (Figure \ref{fig:six_tfiib_spreading_ssa4}).
This difference in binding pattern makes peak calling ineffective for quantifying TFIIB signal in this case: ChIP-seq peak callers generally use different algorithms for calling `narrow' peaks (e.g. for sequence-specific transcription factors) and `broad' peaks (e.g. for histone modifications), meaning that a single algorithm is unable to call peaks that are meaningful for differential binding analyses between wild-type and \textit{spt6-1004}.
Therefore, to see if changes in transcript levels in \textit{spt6-1004} correspond to changes in transcription initiation, we compared the change in TSS-seq signal at TSS-seq peaks in \textit{spt6-1004} to the change in TFIIB ChIP-nexus signal in the window extending 200 bp upstream of the TSS-seq peak.
Changes in TSS-seq signal in \textit{spt6-1004} are associated with a change in TFIIB signal of the same sign at over 82\% of TSSs of any genomic class, indicating that the increase in intragenic transcript levels and decrease in genic transcript levels observed in \textit{spt6-1004} are in large part explained by changes in transcription initiation.

\begin{SCfigure}[50][h]
\centering
\includegraphics[width=3.5in]{figures/six/six_tfiib_spreading_ssa4.pdf}
\caption[TFIIB ChIP-nexus protection over the 20 kb flanking the gene \textit{SSA4}, in wild-type and \textit{spt6-1004} cells.]{
    \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=4.2em, font=\normalfont]
        \item [top)] TFIIB ChIP-nexus protection in wild-type and \textit{spt6-1004}, over 20 kb of chromosome II flanking the \textit{SSA4} gene.
        \item [bottom)] Expanded view of TFIIB protection over the \textit{SSA4} gene.
    \end{description}
}
\label{fig:six_tfiib_spreading_ssa4}
\end{SCfigure}

\begin{figure}[h]
\centering
\includegraphics[width=6in]{figures/six/six_tss_v_tfiib.pdf}
\caption[Scatterplots of fold-change in \textit{spt6-1004} over wild-type, comparing TSS-seq and TFIIB ChIP-nexus.]{Scatterplots of fold-change in \textit{spt6-1004} over wild-type, comparing TSS-seq and TFIIB ChIP-nexus. Each dot represents a TSS-seq peak paired with the window extending 200 bp upstream of the TSS-seq peak summit for quantification of TFIIB ChIP-nexus signal. Fold-changes are regularized fold-change estimates from DESeq2, with size factors determined from the \textit{S. pombe} spike-in (TSS-seq), or \textit{S. cerevisiae} counts (ChIP-nexus).}
\end{figure}

\section{MNase-seq results from \textit{spt6-1004}}

Because a primary function of Spt6 is to act as histone chaperone that reassembles nucleosomes in the wake of transcription \citep{duina2011}, it is reasonable to expect that the transcriptional changes seen in \textit{spt6-1004} would be associated with changes in chromatin structure.
The requirement for Spt6 in maintaining normal chromatin structure has been demonstrated in previous studies \citep{bortvin1996, ivanovska2011, jeronimo2015, kaplan2003, perales2013, vanbakel2013}.
To re-examine this requirement in higher resolution, we assayed nucleosome protection genome-wide using micrococcal nuclease digestion of chromatin followed by sequencing (MNase-seq).

\begin{figure}[H]
    \centering
    \begin{minipage}[t]{2.875in}
        \centering
        \includegraphics[width=2.875in]{figures/six/six_mnase_metagene.pdf}
        \caption[Average MNase-seq dyad signal in wild-type and \textit{spt6-1004}, over non-overlapping genes aligned by wild-type +1 nucleosome dyad.]{Average MNase-seq dyad signal in wild-type and \textit{spt6-1004}, over 3522 non-overlapping genes aligned by wild-type +1 nucleosome dyad. Values are the mean of spike-in normalized coverage in non-overlapping 20 bp bins, averaged over two replicates (\textit{spt6-1004}) or one experiment (wild-type). The solid line and shading are the median and the inter-quartile range.}
        \label{fig:six_mnase_metagene}
    \end{minipage}\hfill
    \begin{minipage}[t]{2.875in}
        \centering
        \includegraphics[width=2.875in]{figures/six/six_global_nuc_occ_fuzz.pdf}
        \caption[Contour plot of nucleosome occupancy and fuzziness in wild-type and \textit{spt6-1004}.]{Contour plot of the global distributions of nucleosome occupancy and fuzziness in wild-type and \textit{spt6-1004}. Dashed lines indicate median values.}
        \label{fig:six_global_nuc_occ_fuzz}
    \end{minipage}
\end{figure}

In wild-type, the MNase-seq data recapitulate the expected signature over genes, with a nucleosome-depleted region upstream of a strongly positioned `+1' nucleosome, and a regularly phased array of nucleosomes over the gene body (Figure \ref{fig:six_mnase_metagene}).
In \textit{spt6-1004}, nucleosome signal is severely reduced at canonical nucleosome positions and spreads into inter-nucleosome regions.
Changes in aggregate nucleosome signal such as those observed in Figure \ref{fig:six_mnase_metagene} are the combination of changes to nucleosome occupancy (the number of reads assigned to a nucleosome), fuzziness (the standard deviation of read positions for a nucleosome), and position (the coordinate with the maximum reads for a nucleosome) \citep{chen2013}.
Using DANPOS2 \citep{chen2013}, we called nucleosome positions and quantified these metrics for wild-type and \textit{spt6-1004}.
Wild-type nucleosomes span a relatively wide range of occupancy and fuzziness space, with highly occupied nucleosomes tending to be less fuzzy (i.e., more well-positioned) (Figure \ref{fig:six_global_nuc_occ_fuzz}).
In \textit{spt6-1004}, the population of nucleosomes is much more homogeneous: nucleosome occupancy is decreased globally, and nucleosome fuzziness is restricted to the high end of the wild-type distribution.

Previous studies observed two trends: 1) In wild-type cells, nucleosome positioning is weaker over highly transcribed genes than over moderately transcribed genes \citep{shivaswamy2008}, and 2) In \textit{spt6-1004} cells, the decrease in nucleosome occupancy is greater for highly transcribed genes \citep{ivanovska2011}.
To re-examine these trends, we looked at the MNase-seq data in the context of NET-seq data, which reports the position of actively transcribing RNAPII and reflects a gene's level of transcription (Figure \ref{fig:six_mnase_heatmaps}) \citep{churchman2012}.
The data support the first trend: in wild-type, genes with the strongest NET-seq signal have decreased MNase-seq signal.
However, there is no obvious relationship between transcription level and the nucleosome changes observed in \textit{spt6-1004} (Figure \ref{fig:six_mnase_heatmaps}).
The apparent discrepancy might be explained by the improved resolution and breadth of MNase-seq versus the MNase and microarray of chromosome III used in the previous study \citep{ivanovska2011}.

\begin{figure}[H]
    \centering
    \includegraphics[width=6in]{figures/six/six_mnase_heatmaps.pdf}
    \caption[Heatmaps of sense NET-seq signal, MNase-seq dyad signal, nucleosome occupancy changes, and nucleosome fuzziness changes over non-overlapping coding genes, aligned by genic TSS and arranged by sense NET-seq signal.]{
        \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=4.2em, font=\normalfont]
            \item [left)] Heatmap of sense strand NET-seq signal for 3522 non-overlapping genes, aligned by genic TSS and sorted by total sense strand NET-seq signal in the window extending 500 nucleotides downstream from the genic TSS. Values are the mean of library-size normalized coverage in non-overlapping 20 nt bins, averaged over two replicates.
            \item [middle)] Heatmaps of MNase-seq dyad signal in wild-type and \textit{spt6-1004} for the same genes, aligned by wild-type +1 nucleosome dyad and arranged by sense NET-seq signal as in the leftmost panel. Values are the mean of spike-in normalized coverage in non-overlapping 20 bp bins, averaged over two replicates (\textit{spt6-1004}) or one experiment (wild-type).
            \item [right)] Heatmaps of fold-change in nucleosome occupancy and fuzziness for the same genes, aligned by wild-type +1 nucleosome dyad and arranged by sense NET-seq signal as in the leftmost panel.
        \end{description}
    }
    \label{fig:six_mnase_heatmaps}
\end{figure}

\subsection{Clustering of MNase-seq profiles at \textit{spt6-1004}-induced intragenic TSSs}

\begin{sidewaysfigure}
    \centering
    \includegraphics[width=8.25in]{figures/six/six_mnase_som.pdf}
    \caption[Average MNase-seq dyad signal around all \textit{spt6-1004}-induced intragenic TSSs, grouped by a self-organizing map of the MNase-seq signal.]{Average MNase-seq dyad signal around all \textit{spt6-1004}-induced intragenic TSSs, grouped by assignment to nodes of a 6x8 super-organizing map (SOM). The number of TSSs assigned to each node is shown in the upper right of each panel, and is shaded by the node's assignment to a cluster determined by agglomerative hierarchical clustering of the nodes. The solid line and shading are the median and inter-quartile range of the mean spike-in normalized coverage over two replicates (\textit{spt6-1004}) or one experiment (wild-type), in non-overlapping 5 bp bins.}
    \label{fig:six_mnase_som}
\end{sidewaysfigure}

\begin{figure}[h]
\centering
\includegraphics[width=6in]{figures/six/six_intragenic_mnase_metagenes.pdf}
\caption[Average wild-type and \textit{spt6-1004} MNase-seq dyad signal and GC content for three clusters of \textit{spt6-1004}-induced intragenic TSSs, as well as wild-type genic TSSs.]{
    \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=6.2em, font=\normalfont]
        \item [top row)] Average MNase-seq dyad signal for \textit{spt6-1004} intragenic TSSs, both aggregated and grouped into three clusters by the wild-type and \textit{spt6-1004} MNase-seq dyad signal flanking the TSS, as well as all genic TSSs detected in wild-type. Values are the mean of spike-in normalized dyad coverage in non-overlapping 10 bp bins, averaged over two replicates (\textit{spt6-1004}) or one experiment (wild-type). The solid line and shading are the median and inter-quartile range.
        \item [bottom row)] Average GC content of the DNA sequence, as above.
    \end{description}
}
\label{fig:six_intragenic_mnase_metagenes}
\end{figure}

\lipsum[1]

\section{Other features of \textit{spt6-1004} intragenic promoters}

\subsection{Information content of intragenic TSSs}

\begin{wrapfigure}[12]{R}{3in}
\centering
\includegraphics[width=3in]{figures/six/six_tss_seqlogos.pdf}
\caption[Sequence logos of TSS-seq reads overlapping genic and intragenic TSS-seq peaks in \textit{spt6-1004}.]{Sequence logos of the information content of TSS-seq reads overlapping genic and intragenic TSS-seq peaks in \textit{spt6-1004}.}
\label{fig:six_tss_seqlogos}
\end{wrapfigure}

\lipsum[1]

\subsection{Sequence motifs enriched at intragenic TSSs}

\begin{wrapfigure}[15]{r}{3in}
\centering
\includegraphics[width=3in]{figures/six/six_intragenic_tata.pdf}
\caption[Kernel density estimate of matches to a consensus TATA-box motif upstream of genic and \textit{spt6-1004}-induced intragenic TSSs.]{Scaled density of occurrences of exact matches to the motif TATAWAWR upstream of TSSs. For each category, a Gaussian kernel density estimate of the positions of motif occurrences is multiplied by the number of motif occurrences in the genomic category and divided by the number of regions in the category.}
\label{fig:six_intragenic_tata}
\end{wrapfigure}

\lipsum[1]

\begin{figure}[]
% \centering
% \includegraphics[width=6in]{figures/six/six_intragenic_mnase_metagenes.pdf}
    \caption[Sequence logos of motifs enriched upstream of \textit{spt6-1004}-induced intragenic and antisense TSSs.]{}
% \label{fig:six_intragenic_mnase_metagenes}
\end{figure}

\section{Summary}

\newpage
\bibliographystyle{apalike}
\begingroup
\singlespacing
\bibliography{references/spt6}
\endgroup
