\chapter{Genomics of transcription elongation factor Spt6}
\label{chapter:six}

\section{Collaborators}

\begin{description}[align=right, labelwidth=5cm, noitemsep]
    \item [Steve Doris] optimized TSS-seq and ChIP-nexus protocols
    \item [] generated TSS-seq and ChIP-nexus libraries
    \item [Olga Viktorovskaya] generated MNase-seq libraries
    \item [Magdalena Murawska] generated NET-seq libraries
    \item [Dan Spatt] Northern, Western, and ChIP experiments
\end{description}

\section{Introduction to Spt6 and intragenic transcription}

The conserved transcription elongation factor Spt6 interacts directly with RNA polymerase II \citep{close2011, diebold2011, liu2011, sdano2017, sun2010, yoh2007}, histones \citep{bortvin1996, mccullough2015}, and another elongation factor called Spn1/Iws1 \citep{diebold2010b, li2018, mcdonald2010}.
The classification of Spt6 as a transcription elongation factor is based on its association with elongating Pol II \citep{andrulis2000, ivanovska2011, kaplan2000, mayer2010, krogan2002}, and its ability to enhance elongation both \textit{in vitro} \citep{endoh2004} and \textit{in vivo} \citep{ardehali2009}, though Spt6 has also been shown to regulate initiation in a small number of cases \citep{adkins2006, ivanovska2011}.
Evidence suggests that as Spt6 travels with elongating Pol II, it acts as a histone chaperone, reassembling nucleosomes after their displacement from DNA due to transcription \citep{duina2011, ivanovska2011}.
Consistent with its histone chaperone function, Spt6 influences chromatin structure \citep{bortvin1996, degennaro2013, ivanovska2011, jeronimo2015, kaplan2003, perales2013, vanbakel2013}; Spt6 is also required for some histone modifications, including H3K36 methylation \citep{carrozza2005, chu2006, yoh2008, youdell2008}, and, in some organisms, H3K4 and H3K27 methylation \citep{begum2012, chen2012, degennaro2013, wang2017, wang2013}.

\begin{wrapfigure}[10]{r}{3in}
    \centering
    \includegraphics[width=3in]{figures/six/six_spt6_western.pdf}
    \caption[Western blot for Spt6 in wild-type and \textit{spt6-1004} cells, at 30\textdegree C and after 80 minutes at 37\textdegree C.]{Western blot for Spt6 in wild-type and \textit{spt6-1004} cells, at 30\textdegree C and after 80 minutes at 37\textdegree C. Spt6 and Dst1 from a spike-in were detected using $\alpha$-FLAG and $\alpha$-Myc antibodies, respectively. The mean $\pm$ standard deviation of three blots are shown below each lane.}
    \label{fig:six_spt6_western}
\end{wrapfigure}

Studies in the yeasts \textit{Saccharomyces cerevisiae} and \textit{Schizosaccharomyces pombe} have previously examined the requirement for Spt6 in normal transcription \citep{cheung2008, degennaro2013, kaplan2003, pathak2018, uwimana2017, vanbakel2013}.
As Spt6 is essential for viability in \textit{S. cerevisiae}, many of these studies use the same temperature-sensitive \textit{spt6} mutant used in this project, \textbf{\textit{spt6-1004}}, which encodes an in-frame deletion of a helix-hairpin-helix domain within Spt6 \citep{kaplan2003}.
When \textit{spt6-1004} cells are shifted from 30\textdegree C to 37\textdegree C for 80 minutes, bulk Spt6 protein levels are depleted to about 20\% of wild-type levels, though cells are still viable  (Figure \ref{fig:six_spt6_western}, \citep{kaplan2003}).
A notable phenotype of the \textit{spt6-1004} mutant is the appearance of \textbf{intragenic transcripts}, transcripts which appear to arise from within protein-coding sequences in both sense and antisense orientations relative to the coding gene (Figure \ref{fig:six_gene_diagram}) \citep{cheung2008, degennaro2013, kaplan2003, uwimana2017}.

\begin{wrapfigure}[8]{r}{3in}
    \centering
    \includegraphics[width=3in]{figures/six/six_gene_diagram.pdf}
    \caption[Diagram of transcript classes.]{Diagram of transcript orientation with respect to coding DNA sequences, for the categories of transcripts referred to in this document.}
    \label{fig:six_gene_diagram}
\end{wrapfigure}
Previous genome-wide measurements of transcript levels in \textit{spt6-1004} relied on tiled microarrays \citep{cheung2008} and RNA sequencing \citep{degennaro2013, uwimana2017}.
Studying intragenic transcription is difficult with these methods, since the signal for an intragenic transcript in the same orientation as the gene it overlaps is convoluted with the signal from the full-length `genic' transcript (Figures \ref{fig:six_gene_diagram}, \ref{fig:six_aat_assay_comparison}) \citep{cheung2008, lickwar2009}.
Identification of intragenic transcription has thus relied on identifying cases where the signal towards the 3$^\prime$ end of a transcript is greater than the signal towards the 5$^\prime$ end.
However, this leads to both false positives, due to the inherent variability of the signal over a transcript, as well as false negatives, due to the requirement of the intragenic transcript to be well-expressed relative to its corresponding genic transcript in order to be identified.
Additionally, these methods are assays of steady-state RNA levels, which makes them unable to distinguish whether the intragenic transcripts observed in \textit{spt6-1004} result from: A) new intragenic transcription initiation in the mutant, B) reduced decay of intragenic transcripts which are rapidly degraded in wild-type, or C) processing of full-length protein-coding RNAs.
New transcription initiation has been shown to be responsible for individual cases of intragenic initiation \citep{kaplan2003}, but this has not previously been studied on a genome-wide scale.
\begin{SCfigure}[40][h]
    \centering
    \includegraphics[width=4.4in]{figures/six/six_aat_assay_comparison.pdf}
    \caption[RNA-seq, TSS-seq, and TFIIB ChIP-nexus signal at the \textit{AAT2} gene, in \textit{spt6-1004} after 80 minutes at 37\textdegree C.]{Sense strand RNA-seq signal, sense strand TSS-seq signal, and TFIIB ChIP-nexus protection at the \textit{AAT2} gene, in \textit{spt6-1004} after 80 minutes at 37\textdegree C.}
    \label{fig:six_aat_assay_comparison}
\end{SCfigure}

To address these challenges to studying intragenic transcription, we applied two genomic assays to \textit{spt6-1004}: transcription start-site sequencing (\textbf{TSS-seq}), and \textbf{ChIP-nexus of TFIIB}, a component of the RNA polymerase II pre-initiation complex (PIC).
TSS-seq sequences the 5$^\prime$ end of capped and polyadenylated RNAs \citep{arribere2013, malabat2015}, allowing separation of intragenic from genic RNA signals and identification of intragenic transcript starts with single-nucleotide resolution (Figure \ref{fig:six_aat_assay_comparison}).
ChIP-nexus is a high-resolution chromatin immunoprecipitation technique, in which the immunoprecipitated DNA is exonuclease digested up to the bases crosslinked with the protein of interest before sequencing \citep{he2015}.
When applied to the PIC component TFIIB, ChIP-nexus reports where transcription initiation is occurring, thus allowing us to determine if intragenic transcripts in \textit{spt6-1004} result from new transcription initiation.

\section{TSS-seq and TFIIB ChIP-nexus results for \textit{spt6-1004}}
\label{sec:six_tss_tfiib}

To study the relationship between Spt6 and transcription, TSS-seq and TFIIB ChIP-nexus libraries were prepared from wild-type and \textit{spt6-1004} cells, after cultures were shifted from 30\textdegree C to 37\textdegree C for 80 minutes.
In wild-type cells, TSS-seq and TFIIB ChIP-nexus recapitulate their expected distributions over the genome: Most TSS signal is restricted to annotated genic TSSs, while most TFIIB signal is localized just upstream of the TSS (Figures \ref{fig:six_tss_seq_heatmaps}, \ref{fig:six_tfiib_heatmap}).
In \textit{spt6-1004}, the signal for both assays infiltrates gene bodies, reflecting widespread intragenic expression of capped and polyadenylated transcripts, and suggesting that new transcription initiation contributes to the intragenic transcription phenotype.
Notably, sense strand TSS-seq signal in \textit{spt6-1004} tends to occur towards the 3$^\prime$ end of genes, while antisense strand TSS-seq signal tends to occur towards the 5$^\prime$ end of genes.

\begin{figure}[H]
\centering
\includegraphics[width=6in]{figures/six/six_tss_seq_heatmaps.pdf}
\caption[Heatmaps of sense and antisense TSS-seq signal from wild-type and \textit{spt6-1004} cells, over non-overlapping coding genes.]{Heatmaps of sense and antisense TSS-seq signal from wild-type and \textit{spt6-1004} cells, over 3522 non-overlapping coding genes aligned by wild-type genic TSS and sorted by annotated transcript length. Data are shown for each gene up to 300 nucleotides 3$^\prime$ of the cleavage and polyadenylation site (CPS), indicated by the white dotted line. Values are the mean of spike-in normalized coverage in non-overlapping 20 nucleotide bins, averaged over two replicates. Values above the 92\textsuperscript{nd} percentile are set to the 92\textsuperscript{nd} percentile for visualization.}
\label{fig:six_tss_seq_heatmaps}
\end{figure}

The TSS-seq data were quantified by peak calling and differential expression analysis, and classified into genomic categories based on their position relative to coding genes.
As suggested by the heatmap visualization (Figure \ref{fig:six_tss_seq_heatmaps}), we detect significant induction of over 4000 intragenic and antisense TSSs in \textit{spt6-1004} (Figure \ref{fig:six_tss_diffexp_summary}).
Compared to previous studies identifying \textit{spt6-1004} intragenic transcription by tiled microarray and RNA-seq \citep{cheung2008, uwimana2017}, we identify intragenic transcription at over 1000 additional genes (Figure \ref{fig:six_intragenic_genes_bvenn}), with the additional information of exact start sites for all identified TSSs.

\begin{SCfigure}[50][h]
    \centering
    \includegraphics[width=4in]{figures/six/six_tfiib_heatmap.pdf}
    \caption[Heatmaps of TFIIB ChIP-nexus protection from wild-type and \textit{spt6-1004} cells, over non-overlapping coding genes]{Heatmaps of TFIIB binding measured by ChIP-nexus, over the same regions shown in Figure \ref{fig:six_tss_seq_heatmaps}. Values are the mean of library-size normalized coverage in non-overlapping 20 bp bins, averaged over two replicates. Values above the 85\textsuperscript{th} percentile are set to the 85\textsuperscript{th} percentile for visualization.}
    \label{fig:six_tfiib_heatmap}
\end{SCfigure}

The TSS-seq data also revealed an unexpected downregulation of most genic TSSs: In this experiment, we detected a significant downregulation to levels below 67\% of wild-type levels at 75\% (3579/4792) of genic TSSs (Figure \ref{fig:six_tss_diffexp_summary}).
As a result of intragenic/antisense induction and genic repression, expression levels in \textit{spt6-1004} of all classes of transcripts become similar to one another (Figure \ref{fig:six_tss_expression_levels}).

\begin{figure}[H]
    \centering
    \begin{minipage}[t]{2.875in}
        \centering
        \includegraphics[width=2.875in]{figures/six/six_tss_diffexp_summary.pdf}
        \caption[Bar plot of the number of TSS-seq peaks in various genomic classes differentially expressed in \textit{spt6-1004} versus wild-type.]{Bar plot of the number of TSS-seq peaks differentially expressed in \textit{spt6-1004} versus wild-type, both after 80 minutes at 37\textdegree C. The height of each bar is proportional to the total number of peaks in the category, including those not found to be significantly differentially expressed.}
        \label{fig:six_tss_diffexp_summary}
    \end{minipage}\hfill
    \begin{minipage}[t]{2.875in}
        \centering
        \includegraphics[width=2.875in]{figures/six/six_intragenic_genes_bvenn.pdf}
        \caption[Set diagram of the number of genes with \textit{spt6-1004}-induced intragenic transcripts reported in \citet{cheung2008}, \citet{uwimana2017}, and our TSS-seq data.]{Set diagram of the number of genes reported to have \textit{spt6-1004}-induced intragenic transcripts using tiled arrays \citep{cheung2008}, RNA-seq \citep{uwimana2017}, and TSS-seq (this work).}
        \label{fig:six_intragenic_genes_bvenn}
    \end{minipage}
\end{figure}

\begin{wrapfigure}[12]{r}{3in}
    \centering
    \includegraphics[width=3in]{figures/six/six_tss_expression_levels.pdf}
    \caption[Violin plots of expression level distributions for genomic classes of TSS-seq peaks in wild-type and \textit{spt6-1004} cells.]{Violin plots of expression level distributions for genomic classes of TSS-seq peaks in wild-type and \textit{spt6-1004}, both after 80 minutes at 37\textdegree C. Normalized counts are the mean of spike-in size factor normalized counts from two replicates.}
    \label{fig:six_tss_expression_levels}
\end{wrapfigure}

The changes in transcript levels in \textit{spt6-1004} observed by TSS-seq correspond with substantial differences in the pattern of TFIIB binding on the genome.
While TFIIB in wild-type binds in discrete peaks within promoter regions, TFIIB in \textit{spt6-1004} binds much more promiscuously, with many loci having TFIIB signal spread over broad regions of the genome (Figure \ref{fig:six_tfiib_spreading_ssa4}).
This difference in binding pattern makes peak calling ineffective for quantifying TFIIB signal in this case: ChIP-seq peak callers generally use different algorithms for calling `narrow' peaks (e.g. for sequence-specific transcription factors) and `broad' peaks (e.g. for histone modifications), meaning that a single algorithm is unable to call a unified set of peaks that is meaningful for differential binding analyses between wild-type and \textit{spt6-1004}.
Therefore, to see if changes in transcript levels in \textit{spt6-1004} correspond to changes in transcription initiation, we compared the change in TSS-seq signal at TSS-seq peaks in \textit{spt6-1004} to the change in TFIIB ChIP-nexus signal in the window extending 200 bp upstream of the TSS-seq peak.
Changes in TSS-seq signal in \textit{spt6-1004} are associated with a change in TFIIB signal of the same sign at over 82\% of TSSs of any genomic class (Figure \ref{fig:six_tss_v_tfiib}), indicating that the increase in intragenic transcript levels and decrease in genic transcript levels observed in \textit{spt6-1004} are in large part explained by changes in transcription initiation.

\begin{figure}[h]
    \centering
    \includegraphics[width=6in]{figures/six/six_tfiib_spreading_ssa4.pdf}
    \caption[TFIIB ChIP-nexus protection over the 20 kb flanking the gene \textit{SSA4}, in wild-type and \textit{spt6-1004} cells.]{
        \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=4.2em, font=\normalfont]
            \item [top)] TFIIB ChIP-nexus protection in wild-type and \textit{spt6-1004}, over 20 kb of\\ chromosome II flanking the \textit{SSA4} gene.
            \item [bottom)] Expanded view of TFIIB protection over the \textit{SSA4} gene.
        \end{description}
    }
    \label{fig:six_tfiib_spreading_ssa4}

    \vspace{0.5em}

    \includegraphics[width=6in]{figures/six/six_tss_v_tfiib.pdf}
    \caption[Scatterplots of fold-change in \textit{spt6-1004} over wild-type, comparing TSS-seq and TFIIB ChIP-nexus.]{Scatterplots of fold-change in \textit{spt6-1004} over wild-type, comparing TSS-seq and TFIIB ChIP-nexus. Each dot represents a TSS-seq peak paired with the window extending 200 bp upstream of the TSS-seq peak summit for quantification of TFIIB ChIP-nexus signal. Fold-changes are regularized fold-change estimates from DESeq2, with size factors determined from the \textit{S. pombe} spike-in (TSS-seq), or \textit{S. cerevisiae} counts (ChIP-nexus).}
    \label{fig:six_tss_v_tfiib}
\end{figure}

\clearpage

\section{MNase-seq results from \textit{spt6-1004}}

Because a primary function of Spt6 is to act as histone chaperone that reassembles nucleosomes in the wake of transcription \citep{duina2011}, it is reasonable to expect that the transcriptional changes seen in \textit{spt6-1004} would be associated with changes in chromatin structure.
The requirement for Spt6 in maintaining normal chromatin structure has been demonstrated in previous studies \citep{bortvin1996, ivanovska2011, jeronimo2015, kaplan2003, perales2013, vanbakel2013, degennaro2013}.
To re-examine this requirement in higher resolution, we assayed nucleosome protection genome-wide using micrococcal nuclease digestion of chromatin followed by sequencing (MNase-seq).

\begin{figure}[h]
    \centering
    \includegraphics[width=5in]{figures/six/six_mnase_metagene.pdf}
    \caption[Average MNase-seq dyad signal in wild-type and \textit{spt6-1004}, over non-overlapping genes aligned by wild-type +1 nucleosome dyad.]{Average MNase-seq dyad signal in wild-type and \textit{spt6-1004}, over 3522 non-overlapping genes aligned by wild-type +1 nucleosome dyad. The solid line and shading are the median and inter-quartile range of the mean spike-in normalized coverage over two replicates (\textit{spt6-1004}) or one experiment (wild-type), in non-overlapping 20 bp bins.}
    \label{fig:six_mnase_metagene}
\end{figure}

\begin{wrapfigure}[11]{r}{3in}
    \centering
    \includegraphics[width=3in]{figures/six/six_global_nuc_occ_fuzz.pdf}
    \caption[Contour plot of nucleosome occupancy and fuzziness in wild-type and \textit{spt6-1004}.]{Contour plot of the distribution of nucleosome occupancy and fuzziness in wild-type and \textit{spt6-1004}. Dashed lines indicate median values.}
    \label{fig:six_global_nuc_occ_fuzz}
\end{wrapfigure}
In wild-type, the MNase-seq data recapitulate the expected signature over genes, with a nucleosome-depleted region upstream of a strongly positioned `+1' nucleosome, and a regularly phased array of nucleosomes over the gene body (Figure \ref{fig:six_mnase_metagene}).
In \textit{spt6-1004}, nucleosome signal is severely reduced at canonical nucleosome positions and spreads into inter-nucleosome regions.
Changes in aggregate nucleosome signal such as those observed in Figure \ref{fig:six_mnase_metagene} are the combination of changes to nucleosome occupancy (the number of reads assigned to a nucleosome), fuzziness (the standard deviation of read positions for a nucleosome), and position (the coordinate with the maximum reads for a nucleosome) \citep{chen2013}.
Using DANPOS2 \citep{chen2013}, we called nucleosome positions and quantified these metrics for wild-type and \textit{spt6-1004}.
Wild-type nucleosomes span a relatively wide range of occupancy and fuzziness space, with highly occupied nucleosomes tending to be less fuzzy (i.e., more well-positioned) (Figure \ref{fig:six_global_nuc_occ_fuzz}).
In \textit{spt6-1004}, the population of nucleosomes is much more homogeneous: nucleosome occupancy is decreased globally, and nucleosome fuzziness is restricted to the high end of the wild-type distribution.

Previous studies observed two trends: 1) In wild-type cells, nucleosome positioning is weaker over highly transcribed genes than over moderately transcribed genes \citep{shivaswamy2008}, and 2) In \textit{spt6-1004} cells, the decrease in nucleosome occupancy is greater for highly transcribed genes \citep{ivanovska2011}.
To re-examine these trends, we looked at the MNase-seq data in the context of NET-seq data, which reports the position of actively transcribing RNAPII and reflects a gene's level of transcription (Figure \ref{fig:six_mnase_heatmaps}) \citep{churchman2012}.
The data support the first trend: in wild-type, genes with the strongest NET-seq signal have weak patterning of MNase-seq signal.
However, we find no obvious relationship between transcription level and the nucleosome occupancy changes observed in \textit{spt6-1004} (Figure \ref{fig:six_mnase_heatmaps}): Genes with the greatest transcription do tend to have lower MNase-seq signal in \textit{spt6-1004}, but this is expected since these genes also have lower MNase-seq signal in wild-type.
The discrepancy with prior work might be explained by the greater resolution and breadth of MNase-seq versus MNase and microarray of chromosome III \citep{ivanovska2011}.

\begin{figure}[H]
    \centering
    \includegraphics[width=6in]{figures/six/six_mnase_heatmaps.pdf}
    \caption[Heatmaps of sense NET-seq signal, MNase-seq dyad signal, nucleosome occupancy changes, and nucleosome fuzziness changes over non-overlapping coding genes, arranged by sense NET-seq signal.]{
        \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=4.2em, font=\normalfont]
            \item [left)] Heatmap of sense strand NET-seq signal for 3522 non-overlapping genes, aligned by genic TSS and sorted by total sense strand NET-seq signal in the window extending 500 nt downstream from the genic TSS. Values are the mean of library-size normalized coverage in non-overlapping 20 nt bins, averaged over two replicates.
            \item [middle)] Heatmaps of MNase-seq dyad signal in wild-type and \textit{spt6-1004} for the same genes, aligned by wild-type +1 nucleosome dyad and arranged by sense NET-seq signal as in the leftmost panel. Values are the mean of spike-in normalized coverage in non-overlapping 20 bp bins, averaged over two replicates (\textit{spt6-1004}) or one experiment (wild-type).
            \item [right)] Heatmaps of fold-change in nucleosome occupancy and fuzziness for the same genes, aligned by wild-type +1 nucleosome dyad and arranged by sense NET-seq signal as in the leftmost panel.
        \end{description}
    }
    \label{fig:six_mnase_heatmaps}
\end{figure}

\subsection{Clustering of MNase-seq profiles at \textit{spt6-1004}-induced intragenic TSSs}

The aggregate MNase-seq dyad signal around all \textit{spt6-1004} intragenic TSSs is aperiodic (Figure \ref{fig:six_intragenic_mnase_metagenes}, top left panel), which occurs as a result of destructive interference from offset nucleosome phasing patterns.
To discover these phasing patterns, we used the wild-type and \textit{spt6-1004} MNase-seq data flanking intragenic TSSs to train a self-organizing map to assign TSSs with similar MNase-seq patterns to nearby nodes in a rectangular grid (Figure \ref{fig:six_mnase_som}).
This allowed us to see that, although there is considerable diversity in the nucleosome pattern surrounding intragenic TSSs, most intragenic TSSs occur in areas between the positions of nucleosome dyads.
By hierarchically clustering the nodes of the self-organizing map, we further grouped intragenic TSSs into three major clusters differing primarily by the phasing of the nucleosome array relative to the TSS, as shown in Figure \ref{fig:six_intragenic_mnase_metagenes}.
In all three clusters, nucleosomes are disrupted to similar levels in \textit{spt6-1004}.

Because GC-poor DNA sequences are nucleosome disfavoring and are known to occur in promoter regions \citep{iyer1995,kaplan2008,tillo2009,zhang2009}, we also examined the GC content surrounding the three clusters of intragenic TSSs.
For all three clusters, the GC content of the DNA drops just upstream of the TSS to a slightly lesser degree than for genic TSSs (Figure \ref{fig:six_intragenic_mnase_metagenes}).

\begin{sidewaysfigure}
    \centering
    \includegraphics[width=8.25in]{figures/six/six_mnase_som.pdf}
    \caption[Average MNase-seq dyad signal around all \textit{spt6-1004}-induced intragenic TSSs, grouped by a self-organizing map of the MNase-seq signal.]{Average MNase-seq dyad signal around all \textit{spt6-1004}-induced intragenic TSSs, grouped by assignment to nodes of a 6x8 super-organizing map (SOM). The number of TSSs assigned to each node is shown in the upper right of each panel, and is shaded by the node's assignment to a cluster determined by agglomerative hierarchical clustering of the nodes. The solid line and shading are the median and inter-quartile range of the mean spike-in normalized coverage over two replicates (\textit{spt6-1004}) or one experiment (wild-type), in non-overlapping 5 bp bins.}
    \label{fig:six_mnase_som}
\end{sidewaysfigure}

\begin{figure}[H]
\centering
\includegraphics[width=6in]{figures/six/six_intragenic_mnase_metagenes.pdf}
\caption[Average wild-type and \textit{spt6-1004} MNase-seq dyad signal and GC content for three clusters of \textit{spt6-1004}-induced intragenic TSSs, as well as wild-type genic TSSs.]{
    \begin{description}[align=right, nosep, itemindent=0pt, leftmargin=6.2em, font=\normalfont]
        \item [left column)] Average MNase-seq dyad signal for \textit{spt6-1004} intragenic TSSs, both aggregated and grouped into three clusters by the wild-type and \textit{spt6-1004} MNase-seq dyad signal flanking the TSS, as well as all genic TSSs detected in wild-type. Values are the mean of spike-in normalized dyad coverage in non-overlapping 10 bp bins, averaged over two replicates (\textit{spt6-1004}) or one experiment (wild-type). The solid line and shading are the median and inter-quartile range.
        \item [right column)] Average GC content of the DNA sequence in a 21 bp window, as above.
    \end{description}
}
\label{fig:six_intragenic_mnase_metagenes}
\end{figure}

\clearpage

\section{Other features of \textit{spt6-1004} intragenic promoters}

MNase-seq indicates that nucleosomes are lost across the entire genome in \textit{spt6-1004}.
However, TSSs observed in \textit{spt6-1004} occur in specific locations, suggesting that loss of nucleosomes is necessary but not sufficient for intragenic transcription, and that additional features such as the drop in GC content at intragenic TSSs (Figure \ref{fig:six_intragenic_mnase_metagenes}) may be required.
The resolution with which we were able to identify intragenic TSSs allowed us to closely examine sequence features that might contribute to intragenic transcription.

\subsection{Information content and sequence preference}

\begin{wrapfigure}[13]{R}{3in}
\centering
\includegraphics[width=3in]{figures/six/six_tss_seqlogos.pdf}
\caption[Sequence logos of TSS-seq reads overlapping genic and intragenic TSS-seq peaks in \textit{spt6-1004}.]{Sequence logos depicting information content and sequence preference of TSS-seq reads overlapping genic and intragenic TSS-seq peaks in \textit{spt6-1004}.}
\label{fig:six_tss_seqlogos}
\end{wrapfigure}

To examine the DNA sequence preference of TSSs in \textit{spt6-1004}, we aligned the sequences of all TSS-seq reads overlapping TSS-seq peaks of each class, and calculated the information content and sequence distribution for each class.
Intragenic TSSs have a sequence preference almost identical to the previously observed sequence preference of genic TSSs (Figure \ref{fig:six_tss_seqlogos}) \citep{malabat2015}, suggesting that RNA polymerase initiates transcription similarly at genic and intragenic TSSs.

\subsection{Enrichment of the TATA box}

A characteristic feature of canonical genic promoters is the presence of a TATA box or TATA-like DNA element which allows for the recruitment of Pol II and general transcription factors via binding of the TFIID complex, which includes TATA-binding protein \citep{rhee2012}.
To examine whether the presence of TATA elements might contribute to \textit{spt6-1004} intragenic transcription, we looked for exact matches to the TATA consensus sequence TATAWAWR in the window extending 200 nucleotides upstream of \textit{spt6-1004} TSSs, finding matches at 13.7\% of regions upstream of intragenic TSSs and 24.7\% for antisense TSSs, versus 24.4\% for all genic TSSs and 8.9\% for random locations in the genome.
Moreover, the TATA elements found near intragenic and antisense TSSs are highly concentrated in the region 50 to 100 nucleotides upstream of the TSS, where TATA elements are most often found for genic TSSs (Figure \ref{fig:six_intragenic_tata}).
This further supports the model that \textit{spt6-1004} intragenic promoters are sequences similar to canonical genic promoters, which become accessible for transcription initiation when the normal chromatin state is disturbed.

\begin{SCfigure}[50][h]
    \centering
    \includegraphics[width=3.7in]{figures/six/six_intragenic_tata.pdf}
    \caption[Kernel density estimate of matches to a consensus TATA-box motif upstream of genic and \textit{spt6-1004}-induced intragenic TSSs.]{Scaled density of occurrences of exact matches to the motif TATAWAWR upstream of TSSs. For each category, a Gaussian kernel density estimate of the positions of motif occurrences is scaled by the number of motif occurrences per region.}
    \label{fig:six_intragenic_tata}
\end{SCfigure}

\subsection{Sequence motifs discovered}

To discover additional sequence features of \textit{spt6-1004} intragenic promoters, we performed \textit{de novo} motif discovery using MEME-ChIP \citep{machanick2011} for the regions -100 to +30 nucleotides relative to TSS summits.
The most enriched motif found by MEME at both intragenic and antisense \textit{spt6-1004} TSSs is, with respect to sense genic transcription, a GA-rich motif with 3-nucleotide periodicity (Figure \ref{fig:six_meme_motifs}).
This motif occurs at only a small subset of intragenic TSSs, but is highly unlikely to occur by chance (compare the expected to observed number of occurrences in Figure \ref{fig:six_meme_motifs}).
The motif is not enriched at genic TSSs upregulated in \textit{spt6-1004}, and is not an obvious match to a DNA-binding factor in the databases searched \citep{deboer2011,macisaac2006,newburger2008,pachkov2013,teixeira2017,zhu1999,weirauch2014}.
If this motif is directly related to intragenic transcription, we speculate that it might create a DNA structure favorable for transcription initiation.

\begin{figure}[h]
    \centering
    \includegraphics[width=6in]{figures/six/six_meme_motifs.pdf}
    \caption[Sequence logos of motifs discovered by MEME upstream of \textit{spt6-1004}-induced intragenic and antisense TSSs.]{Sequence logos of motifs discovered by MEME \citep{bailey2015} in the window -100 to +30 bp relative to \textit{spt6-1004} intragenic and antisense TSSs. The number of motif occurrences and the E-value, indicating the expected number of motif occurrences if the input sequences were scrambled, are shown for each motif.}
    \label{fig:six_meme_motifs}
\end{figure}

\section{Discussion}

In this work, we integrated multiple quantitative genomic approaches to study the conserved transcription elongation factor Spt6.
Our TSS-seq and TFIIB ChIP-nexus results reveal the full extent of intragenic and antisense transcript expression in \textit{spt6-1004}, and show that these transcripts are largely explained by new RNA Pol II transcription initiation.
Our MNase-seq results show that this new transcription initiation happens in the context of a global depletion and disordering of nucleosomes from chromatin.
We speculate that this dramatic decrease in nucleosome protection of the genome leads to intragenic transcription by allowing initiation factors to access normally inaccessible promoter-like sequences within coding sequences.
This model is supported by the similarities we observe between genic and intragenic promoters in DNA GC content, initiation motif, and TATA element frequency.
This may also explain the unexpected decrease in transcription initiation we see at almost all genic promoters in \textit{spt6-1004}: Assuming that the pool of transcription initiation factors in the cell is limiting, then making thousands of additional binding sites available to the initiation machinery would decrease the frequency at which the initiation machinery finds its correct targets at genic promoters.

\clearpage
\section{Methods}

\subsection{Yeast strain construction and growth conditions}

All yeast strains were constructed by standard yeast transformation or crosses.
The \textit{spt6-1004} and wild-type strains were grown as previously described \citep{cheung2008}: Cells were grown in YPD at 30\textdegree C to a concentration of approximately $1 \times 10^7$ cells/ml (OD$_{600} = 0.6$), at which point an equal volume of YPD medium pre-warmed to 44\textdegree C was added, and the cultures were shifted to 37\textdegree C for 80 minutes.

\subsection{Western blotting}

The protocols for western blotting and quantification are described in \citet{doris2018}.

\subsection{Sequencing library preparation\\(TSS-seq, ChIP-nexus, MNase-seq, NET-seq)}

All library preparation methods are detailed in \citet{doris2018}.

\subsection{Genome builds}

The genome build used for \textit{S. cerevisiae} was R64-2-1 \citep{engel2014}, and the genome build used for \textit{S. pombe} was ASM294v2 \citep{wood2002}.

\subsection{TSS-seq data analysis}
\label{subsec:tss_seq}

An up-to-date version of the Snakemake \citep{koster2012} workflow used to process TSS-seq libraries is maintained at \href{https://github.com/winston-lab/tss-seq}{github.com/winston-lab/tss-seq}.
At the time of writing, removal of adapter sequences and random hexamer sequences from the 3$^\prime$ end of the read and 3$^\prime$ quality trimming were performed using cutadapt \citep{martin2011}.
The random hexamer molecular barcode on the 5$^\prime$ end of the read was then removed and processed using a custom Python script (adapted from \citet{mayer2015}).
Reads were aligned to the combined \textit{S. cerevisiae} and \textit{S. pombe} reference genomes using Tophat2 \citep{kim2013} without a reference transcriptome, and uniquely mapping reads were selected using SAMtools \citep{li2009}.
Reads mapping to the same location as another read with the same molecular barcode were identified as PCR duplicates and removed using a custom Python script (adapted from \citet{mayer2015}).
Coverage of the 5$^\prime$-most base, corresponding to the TSS, was extracted using bedtools genomecov \citep{quinlan2010} and normalized to the total number of uniquely mapping, non-duplicate \textit{S. pombe} alignments.
Quality statistics of raw, cleaned, non-aligning, and uniquely aligning non-duplicate reads were assessed using FastQC \citep{andrews2010}.

The pipeline additionally performs \hyperref[subsubsec:tss_peak_calling]{TSS-seq peak calling}, \hyperref[subsubsec:tss_differential_expression]{differential expression}, \hyperref[subsubsec:tss_peak_classification]{classification of peaks into genomic categories}, \hyperref[subsubsec:tss_seqlogos]{sequence logo visualization}, motif enrichment analysis, \hyperref[subsubsec:denovo_motif_discovery]{\textit{de novo} motif discovery}, gene ontology analysis \citep{young2010}, and data visualization with the option to separate data into clusters of similar signal.

\subsubsection{Reannotation of \textit{S. cerevisiae} TSSs using TSS-seq data}

TSS-seq coverage from two replicates of a wild-type \textit{S. cerevisiae} strain grown at 30\textdegree C in YPD was averaged and used to adjust the 5$^\prime$ ends of an annotation of major transcript isoforms based on TIF-seq data \citep{pelechano2013}.
The 5$^\prime$ end of the original annotation was changed to the position of maximum TSS-seq signal in a window $\pm$ 250 nt of the original 5$^\prime$ end if the maximum TSS-seq signal was greater than the 95\textsuperscript{th} percentile of all non-zero TSS-seq signal.

\subsubsection{TSS-seq peak calling}
\label{subsubsec:tss_peak_calling}

TSS-seq data representing transcription from a single promoter tends to occur as a cluster of signal distributed over a range of positions, rather than a single nucleotide \citep{arribere2013, malabat2015}.
It is reasonable to consider such a cluster of TSS-seq signal as a single entity, because the signals within the cluster are usually highly correlated to one another across different conditions.
Therefore, to identify TSSs from TSS-seq data and quantify them for downstream analyses such as differential expression, it is necessary to annotate these groups of TSS-seq signal by using the data to perform peak-calling.

At the time of writing, TSS-seq peak calling for a given experimental group was performed by 1-D watershed segmentation of the data for each sample in the group, followed by filtering for reproducibility within the group by the Irreproducible Discovery Rate (IDR) method \citep{li2011}.
First, a smoothed version of the TSS-seq coverage is generated for each sample using an adaptive two-stage kernel density estimation with a discretized Gaussian kernel \citep{silverman1986}.
For a given nucleotide, the adaptive kernel bandwidth, $\sigma_{\text{adaptive}}$, is given by
\begin{align*}
    \sigma_{\text{adaptive}} &= \sigma_\text{pilot} \left( \frac{\rho_{\text{pilot}}}{g} \right)^{-\alpha},
\end{align*}
where $\sigma_\text{pilot}$ is the standard, fixed bandwidth of a Gaussian kernel used to calculate the pilot signal density $\rho_\text{pilot}$ at that nucleotide, $g$ is the geometric mean of $\rho_\text{pilot}$ over the whole genome, and $\alpha$ is a parameter in $[0,1]$ that determines the degree to which the pilot density $\rho_\text{pilot}$ affects $\sigma_\text{adaptive}$.
The adaptive kernel adjusts the kernel bandwidth to be smaller in regions of high signal density and larger in regions of lower signal density, allowing the smoother to better accommodate both `sharp' TSSs where the signal is distributed over a relatively small window, as well as `broad' TSSs where the signal is more dispersed.
For all analyses in this document, adaptive smoothing was performed with $\sigma_\text{pilot} = 10$ and $\alpha = 0.2$.

Following smoothing, an initial set of peaks is formed by assigning all nonzero signal in the original, unsmoothed coverage to the nearest local maximum of the smoothed coverage, and taking the minimum and maximum genomic coordinates of the original coverage as the peak boundaries for each local maximum of the smoothed coverage.
Peaks are then trimmed to the smallest genomic interval that includes 95\% of the original coverage, and the probability of the peak begin generated by noise is estimated by a Poisson model where $\lambda$, the expected coverage, is the maximum of the expected coverage over the chromosome and the expected coverage in the 2 kb window upstream of the peak (\`a la the ChIP-seq peak caller MACS2 \citep{zhang2008}).
Finally, peaks are ranked by their significance under the Poisson model, and a final list of peaks for the group is generated using the IDR method ($\text{IDR}=0.1$) \citep{li2011}.
In brief, IDR compares ranked lists of regions in order to set a cutoff, beyond which the regions are no longer consistent between replicates.

The python script used for 1-D watershed segmentation of TSS-seq data is \href{https://github.com/winston-lab/tss-seq/blob/master/scripts/tss_peakcalling.py}{available as part of the TSS-seq pipeline}, and the IDR implementation used in the pipeline is also \href{https://github.com/nboley/idr}{available on GitHub}.

\subsubsection{TSS differential expression analysis}
\label{subsubsec:tss_differential_expression}

For TSS-seq differential expression analysis, TSS-seq peak-calling was performed \hyperref[subsubsec:tss_peak_calling]{as described above} for both \textit{S. cerevisiae} and the \textit{S. pombe} spike-in.
The read counts for each peak in each condition were used as the input to differential expression analysis by DESeq2 \citep{love2014}, with the alternative hypothesis $\allowbreak \left\lvert\log_2 \left(\text{fold-change}\right) \right\rvert > 1.5$ and a false discovery rate of 0.1.
To normalize by spike-in, the size factors of the \textit{S. pombe} spike-in counts were used as the size factors for \textit{S. cerevisiae}, although we note that due to the median of ratios normalization used in DESeq2, the major TSS-seq results of this work are still observed when \textit{S. cerevisiae} size factors are used.

\subsubsection{Classification of TSS-seq peaks into genomic categories}
\label{subsubsec:tss_peak_classification}

TSS-seq peaks were assigned to genomic categories based on their position relative to the transcript annotation \hyperref[subsubsec:tss_reannotation]{described above} and an annotation of all verified open reading frames (ORFs) and blocked reading frames in \textit{S. cerevisiae} \citep{engel2014}.
First, `genic' regions were defined as follows: If a gene was present in both the transcript and ORF annotations, the genic region was defined as the interval (annotated TSS-30 nt, start codon).
If gene was present in the transcript annotation but not the ORF annotation, the genic region was defined as the interval (annotated TSS - 30 nt, annotated TSS + 30 nt).
If a gene was present only in the ORF annotation, the genic region was defined as the interval (start codon - 30 nt, start codon).
For the purposes of peak classification, regions were considered overlapping if they had at least one base of overlap.
TSS-seq peaks were classified as genic if they overlapped a genic region on the same strand.
Peaks were classified as intragenic if they were not classified as a genic peak, and their summit position overlapped an open or closed reading frame on the same strand.
Peaks were classified as antisense if their summit position overlapped a transcript on the opposite strand.
Finally, peaks were classified as intergenic if they did not overlap a transcript, reading frame, or genic region on either strand.

\subsubsection{TSS information content and sequence composition}
\label{subsubsec:tss_seqlogos}

TSS-seq alignments were pooled for all replicates in a condition, and the DNA sequence flanking the position of every read overlapping TSS-seq peaks of a particular genomic category was extracted using SAMtools \citep{li2009} and bedtools \citep{quinlan2010}.
The information content and sequence composition of the sequences was quantified using WebLogo \citep{crooks2004}, with the zeroth-order Markov model of the \textit{S. cerevisiae} genomic sequence as the background composition.
Sequence logos were plotting using helper functions from ggseqlogo \citep{wagih2017}.

\subsubsection{Enrichment of the TATA box}

An up-to-date version of the Snakemake \citep{koster2012} workflow used to test the enrichment of motifs is maintained at \href{https://github.com/winston-lab/motif-enrichment}{github.com/winston-lab/motif-\\enrichment}.
To test for enrichment of consensus TATA boxes, FIMO \citep{grant2011} was used to search the \textit{S.cerevisiae} genome for matches to the query motif TATAWAWR (where the ambiguous bases are equiprobable) at a p-value threshold of $6 \times 10^{-4}$.
Regions extending 200 nucleotides upstream of TSS summits were merged if they were overlapping, and were considered to contain a consensus TATA box if the entire motif was overlapping the region on the same strand.
The frequency of motif occurrences in the regions of interest was compared to the frequency of occurrences in the regions upstream of 6000 randomly chosen locations in the genome, using Fisher's exact test.

\subsubsection{\textit{De novo} motif discovery}
\label{subsubsec:denovo_motif_discovery}

\textit{De novo} motif discovery for the regions around TSSs was performed by running MEME-ChIP \citep{machanick2011} on the DNA sequence -100 to +30 nucleotides from the TSS summits of the \hyperref[subsubsec:tss_peak_classification]{genomic classes} of TSSs \hyperref[subsubsec:tss_differential_expression]{significantly upregulated} in \textit{spt6-1004} versus wild-type.

\subsection{ChIP-nexus data analysis}
An up-to-date version of the Snakemake \citep{koster2012} workflow used to process ChIP-nexus libraries is maintained at \href{https://github.com/winston-lab/chip-nexus}{github.com/winston-lab/chip-nexus}.
At the time of writing, filtering for reads containing the constant region of the adapter on the 5$^\prime$ end of the read, 3$^\prime$ adapter removal, and 3$^\prime$ quality trimming were performed using cutadapt \citep{martin2011}.
The random pentamer molecular barcode on the 5$^\prime$ end of the read was then removed and processed using a custom Python script modified from \citet{mayer2015}.
Reads were aligned to the combined \textit{S. cerevisiae} and \textit{S. pombe} genomes using Bowtie 2 \citep{langmead2012}, and uniquely mapping alignments were selected using SAMtools \citep{li2009}.
Reads mapping to the same location as another read with the same molecular barcode were identified as PCR duplicates and removed using a custom Python script modified from \citet{mayer2015}.
Coverage of the 5$^\prime$-most base, corresponding to the point of crosslinking, was extracted using bedtools genomecov \citep{quinlan2010}.
The median fragment size estimated by MACS2 \citep{zhang2008} over all samples was used to generate coverage of factor protection and fragment midpoints, by extending reads to the fragment size, or by shifting reads by half the fragment size, respectively.
Coverage was normalized to the total number of reads uniquely mapping to \textit{S. cerevisiae} (the \textit{S. pombe} spike-in was not used for normalization due to the low number of reads mapping to \textit{S. pombe}).
Quality statistics of raw, cleaned, non-aligning, and uniquely aligning non-duplicate reads were assessed using FastQC \citep{andrews2010}.

The ChIP-nexus pipeline additionally performs \hyperref[subsubsec:nexus_peak_calling]{peak calling}, \hyperref[subsubsec:nexus_differential_occupancy]{differential occupancy analysis}, and data visualization with the option to separate data into clusters of similar signal.

An second Snakemake workflow for TFIIB-specific analyses is maintained at\\\href{https://github.com/winston-lab/chip-nexus-tfiib}{github.com/winston-lab/chip-nexus-tfiib}, and performs \hyperref[subsubsec:tfiib_peak_classification]{classification of TFIIB peaks into genomic categories}, motif enrichment analysis, and gene ontology analysis.

\subsubsection{ChIP-nexus peak calling}
\label{subsubsec:nexus_peak_calling}

A number of tools have been created specifically for peak-calling using data from high-resolution ChIP techniques such as ChIP-nexus and ChIP-exo \citep{wang2014, hansen2016}.
When applied to our TFIIB ChIP-nexus data, these tools tended to split what appeared to be a single TFIIB binding event into multiple peaks.
This may be because TFIIB crosslinks to DNA at multiple points \citep{rhee2012}, suggesting that while these tools may work well for factors that bind symmetrically to DNA with a single crosslinking point on either side, there is room for improvement for factors with more complex crosslinking patterns.

The ChIP-nexus pipeline currently performs peak calling for a condition using the standard ChIP-seq peak caller MACS2 \citep{zhang2008}, followed by filtering for reproduciblity by the Irreproducible Discovery Rate (IDR) method (IDR = 0.1 for all analyses in this chapter) \citep{li2011}.

\subsubsection{TFIIB ChIP-nexus differential occupancy analysis}
\label{subsubsec:nexus_differential_occupancy}
For TFIIB ChIP-nexus differential binding analysis, TFIIB peaks were called by MACS2 and IDR filtering \hyperref[subsubsec:tfiib_peak_calling]{as described above}.
A non-redundant list of peaks called in the condition and control groups being compared was generated using bedtools multiinter \citep{quinlan2010}, and the counts of fragment midpoints for each peak in each sample were used as the input to differential binding analysis by DESeq2 \citep{love2014}, with the alternative hypothesis $\allowbreak \left\lvert\log_2 \left(\text{fold-change}\right) \right\rvert > 1.5$ and a false discovery rate of 0.1.
For estimation of change in TFIIB binding upstream of TSS-seq peaks, TFIIB fragment midpoint counts in the window extending 200 bp upstream of the TSS-seq peak summit were used as the input to DESeq2.
\textit{S. cerevisiae} counts were used for size factor calculation.

\subsubsection{Classification of TFIIB ChIP-nexus peaks into genomic categories}
\label{subsubsec:tfiib_peak_classification}
As for TSS-seq peaks, TFIIB ChIP-nexus peaks were assigned to genomic categories based on their position relative to the transcript annotation \hyperref[subsubsec:tss_reannotation]{described above}, an annotation of all verified open reading frames (ORFs) and blocked reading frames \citep{engel2014}, and \hyperref[subsubsec:tss_peak_classification]{an annotation of `genic' regions derived from the transcript and ORF annotations}.
TFIIB ChIP-nexus peaks were classified as genic if they overlapped a genic region.
Peaks were classified as intragenic if they were not classified as a genic peak, and the entire peak overlapped an open or closed reading frame.
Finally, peaks were classified as intergenic if they did not overlap a transcript, reading frame, or genic region.

\subsection{Comparison of TSS-seq to TFIIB ChIP-nexus}

An up-to-date version of the Snakemake \citep{koster2012} workflow used to compare TSS-seq data to TFIIB ChIP-nexus data is maintained at\\\href{https://github.com/winston-lab/tss-seq-vs-tfiib-nexus}{github.com/winston-lab/tss-seq-vs-tfiib-nexus}.
The pipeline matches and compares peaks from the two assays, and also performs the TFIIB differential occupancy analysis over windows upstream of TSS-seq peaks shown in section \ref{sec:six_tss_tfiib} and described in section \ref{subsubsec:nexus_differential_occupancy}.

\subsection{MNase-seq data analysis}
\label{subsec:mnase_seq}

An up-to-date version of the Snakemake \citep{koster2012} workflow used to demultiplex paired-end MNase-seq libraries is maintained at \href{https://github.com/winston-lab/demultiplex-paired-end}{github.com/winston-lab/demultiplex-paired-end}.
At the time of writing, demultiplexing was performed using fastq-multx \citep{aronesty2013}, allowing one mismatch to the barcode, followed by filtering for and removal of the barcode on read 2 using cutadapt \citep{martin2011}.

An up-to-date version of the Snakemake \citep{koster2012} workflow used to process MNase-seq libraries is maintained at \href{https://github.com/winston-lab/mnase-seq}{github.com/winston-lab/mnase-seq}.
At the time of writing, 3$^\prime$ quality trimming was performed using cutadapt \citep{martin2011}.
Reads were aligned to the combined \textit{S. cerevisiae} and \textit{S. pombe} genome using Bowtie 1 \citep{langmead2009}, and correctly paired alignments were selected using SAMtools \citep{li2009}.
Coverage of nucleosome protection and nucleosome dyads were extracted using bedtools \citep{quinlan2010} and custom shell scripts to get the entire fragment or the midpoint of the fragment, respectively.
Smoothed nucleosome dyad coverage was generated by smoothing dyad coverage with a Gaussian kernel of 20 bp bandwidth.
Coverage was normalized to the total number of correctly paired \textit{S. pombe} fragments.
Quality statistics of raw, cleaned, non-aligning, and correctly paired reads were assessed using FastQC \citep{andrews2010}.

The MNase-seq pipeline additionally performs \hyperref[subsubsec:nucleosome_quantification]{quantification of nucleosome properties}, and data visualization with the option to separate data into clusters of similar signal.

\subsubsection{Quantification of nucleosome properties}
\label{subsubsec:nucleosome_quantification}
Quantification of nucleosome occupancy, fuzziness, and position shifts were calculated using DANPOS2 \citep{chen2013}, with spike-in normalization by scaling the total counts in condition group libraries by
\begin{align*}
    \frac{\text{mean observed percent spike-in in condition libraries}}{\text{mean observed percent spike-in in control libraries}}.
\end{align*}

\subsubsection{Clustering of MNase-seq signal at \textit{spt6-1004} intragenic TSSs}

The Snakemake \citep{koster2012} workflow for clustering MNase-seq data by self/super-organizing map and hierarchical clustering is maintained at\\\href{https://github.com/winston-lab/cluster-mnase-seq}{github.com/winston-lab/cluster-mnase-seq}.
To cluster \textit{spt6-1004} intragenic TSSs based on surrounding MNase-seq signal, spike-in normalized MNase-seq dyad signal in the window $\pm150$ bp of the TSS-seq peak summit of all intragenic TSS-seq peaks significantly upregulated in \textit{spt6-1004} was binned by taking the mean signal in non-overlapping 5 bp bins, and then averaged taking the mean of two replicates (\textit{spt6-1004}) or one experiment (wild-type).
The data were then standardized over each TSS, and the wild-type and \textit{spt6-1004} data were used as equally weighted input layers to a super-organizing map \citep{wehrens2007} trained on the input data to assign similar MNase-seq observations in 60-dimensional input space to similar nodes in a 2-dimensional ($6 \times 8$) rectangular grid.
The 48 `code vectors' representing the typical MNase-seq pattern for each node (visualized in Figure \ref{fig:six_mnase_som}) were then clustered by agglomerative hierarchical clustering using sum of squares distance and Ward linkage.
The resulting dendrogram was cut to produce the three clusters of MNase-seq signal shown in Figures \ref{fig:six_mnase_som} and \ref{fig:six_intragenic_mnase_metagenes}.

\subsection{NET-seq data analysis}
\label{subsec:net_seq}

An up-to-date version of the Snakemake \citep{koster2012} workflow used to process NET-seq libraries is maintained at \href{https://github.com/winston-lab/net-and-rna-seq}{github.com/winston-lab/net-and-rna-seq}.

At the time of writing, removal of adapter sequences from the 3$^\prime$ end of the read and 3$^\prime$ quality trimming were performed with cutadapt \citep{martin2011}.
Reads were aligned to the \textit{S. cerevisiae} genome using Tophat2 \citep{kim2013} without a reference transcriptome, and uniquely mapping reads were selected using SAMtools \citep{li2009}.
Coverage of the 5$^\prime$-most base of the read, corresponding to the 3$^\prime$-most base of the nascent RNA and the active site of elongating RNA polymerase, was extracted using bedtools genomecov \citep{quinlan2010} and normalized to the total number of uniquely mapped reads.
Quality statistics of raw, cleaned, non-aligning, and uniquely aligning reads were assessed using FastQC \citep{andrews2010}.

The NET-seq pipeline additionally performs \textit{ab initio} transcript annotation \citep{pertea2015}, differential expression analysis, and data visualization with the option to split data into clusters of similar signal.
For libraries with unique molecular barcodes and/or spike-ins, the pipeline also handles PCR duplicate removal and spike-in normalization, respectively.

\clearpage
\bibliographystyle{apalike}
\begingroup
    \singlespacing
    \bibliography{references/spt6}
\endgroup
