\chapter{Studies of the functions of intragenic transcription in stress}
\label{chapter:stress}

\section{Abstract}

Intragenic transcription opens up interesting possibilities for gene regulation in wild-type cells, and individual cases of intragenic transcription have previously been shown to be important for different biological processes.
To discover uncharacterized instances of intragenic transcription, we applied high resolution genomic assays of transcription initiation to wild-type \textit{Saccharomyces cerevisiae} under three stress conditions.
For the condition of oxidative stress, we show that intragenic transcripts are generally expressed at lower levels than genic transcripts, and that many intragenic transcripts are likely to be translated at some level.
By comparing intragenic transcription in three yeast species, we find that most examples of oxidative-stress-regulated intragenic transcription identified in \textit{S. cerevisiae} are not conserved.
Finally, we show that the expression of an oxidative-stress-induced intragenic transcript at the gene \textit{DSK2} is needed for survival in conditions of oxidative stress.

\clearpage

\section{Collaborators}

\begin{description}[align=right, labelwidth=5cm, noitemsep, leftmargin=!]
    \item [Steve Doris] generated TSS-seq and ChIP-nexus libraries
    \item [Dan Spatt] polyribosome fractionation, competitive growth assays, and Northern blots
    \item [James Warner] Northern blots
\end{description}

\section{Functions of intragenic transcription in wild-type cells}

In chapters \ref{chapter:six} and \ref{chapter:five}, we presented examples in which mutation of certain transcription elongation factors leads to the expression of intragenic transcripts, i.e., transcripts initiating from within gene bodies.
This implies that wild-type cells have evolved to prevent the occurrence of spurious intragenic transcription.
Despite this, our studies and others \citep{cheung2008, doris2018, malabat2015, pelechano2013a} consistently detect some instances of intragenic transcription in wild-type cells, some of which could have biological functions.

One mechanism by which intragenic transcription can exert a function is by being translated into a functional polypeptide or protein.
Translation of a sense-strand intragenic transcript is likely to lead to production of an N-terminally truncated protein, which may differ from the full-length protein in stability, subcellular localization, or function due to the absence of domains normally found in the full-length protein (a few examples include \citet{carlson1982, benanti2009, gammie1999, mcknight2014}).
N-terminally truncated proteins can even exert dominant negative effects: In \textit{S. cerevisiae} cells under DNA replication stress, intragenic transcription of the mitotic spindle microtubule bundling gene \textit{ASE1} leads to expression of an Ase1 protein which retains a C-terminal microtubule-binding domain but lacks an N-terminal dimerization domain \citep{mcknight2014}.
Evidence suggests that by binding microtubules but not dimerizing, intragenic Ase1 antagonizes the function of full-length Ase1 and allows for the proper cellular response to replicative stress.

In other cases, it may be the process of intragenic transcription that is important, rather than the RNA produced.
As transcription occurs, co-transcriptional processses including chromatin remodeling and histone modification also take place via the recruitment of various factors to the transcription complex.
These factors are often recruited in a manner dependent on the stage of transcription, which results in stereotypic patterns of chromatin state over transcribed regions \citep{smolle2013, buratowski2010}.
For example, in \textit{S. cerevisiae}, the Set1 complex responsible for methylation of histone H3 lysine 4 (H3K4) is recruited to Pol II specifically during early elongation, resulting in a pattern where H3K4 trimethylation (H3K4me3) is primarily found over the 5$^\prime$ ends of transcribed regions \citep{liu2005, barski2007, pokholok2005, soares2017}.
Since intragenic transcribed regions overlap genic transcribed regions, intragenic transcription can generate unusual patterns of nucleosome positioning and histone modifications over a gene, which may influence expression from the genic promoter.
One example of this occurs at the \textit{S. cerevisiae} asparagine catabolic gene \textit{ASP3}, where sense-strand intragenic transcription is necessary for wild-type levels of H3K4me3 over the 5$^\prime$ end of the gene and full induction of the gene upon nitrogen stress \citep{huang2010}.

Even more possibilities for gene regulation by intragenic transcription have been revealed by studies of antisense transcription (reviewed in \citet{pelechano2013b}): Antisense transcription can physically block sense transcription, and antisense transcripts can base pair with sense RNAs to influence isoform selection, stability, and translation efficiency of the sense transcript.
By base pairing with DNA, sense or antisense intragenic RNAs can also form R-loop structures with the potential to inhibit transcription or regulate processes like DNA methylation.

We seek to better understand some of the many ways in which intragenic transcription can function in cells by investigating uncharacterized instances of intragenic transcription.
However, this is complicated by the possibility that some cases of intragenic transcription have no real function.
Because the sequence of a gene is constrained by the requirement to encode a functional protein, a fortuitously formed intragenic promoter could potentially be evolutionarily maintained in the absence of intragenic function.
Therefore, with the reasoning that intragenic transcription which is regulated by an environmental perturbation is more likely to have a function, we decided to identify stress-regulated intragenic transcription in wild-type \textit{S. cerevisiae}.

% ASE1 \citep{mcknight2014}.
% KAR4 \citep{gammie1999}.
% CIK1 \citep{benanti2009}.
% ASP3 \citep{huang2010}.

\section[Discovery of stress-regulated intragenic promoters\\ by TFIIB ChIP-nexus and TSS-seq]{Discovery of stress-regulated intragenic promoters by TFIIB ChIP-nexus and TSS-seq}

To discover cases of stress-regulated intragenic transcription initiation, we performed ChIP-nexus of TFIIB for wild-type yeast under conditions of oxidative stress, amino acid stress, and nitrogen stress, along with controls of growth in rich YPD medium and defined SC medium.
The genic TFIIB response to each of the stresses either correlates well with the expected transcriptomic response to the stress (Figure \ref{fig:stress_gasch_comparison}), or is enriched for metabolic pathways consistent with the cellular response to the stress (Figure \ref{fig:stress_nitrogen_gene_ontology}), confirming that TFIIB ChIP-nexus captures changes in transcription initiation and that the cells were stressed as intended.
In total, we identify 140 intragenic TFIIB peaks significantly induced at least 1.5-fold in at least one stress condition, with some peaks being induced in more than one stress (Figures \ref{fig:stress_tfiib_coverage}, \ref{fig:stress_tfiib_ridgelines}).
We also observe a slight anti-correlation between stress-induced changes in intragenic TFIIB signal and changes in the corresponding genic TFIIB signal (Figure \ref{fig:stress_genic_vs_intra}).
\begin{figure}[h]
    \centering
    \includegraphics[width=6in]{figures/stress/stress_gasch_comparison.pdf}
    \caption[Scatterplots comparing change in genic TFIIB signal to change in RNA microarray signal, for oxidative and amino acid stresses.]{Scatterplots comparing change in genic TFIIB signal to change in RNA microarray signal from \citet{gasch2000}, for oxidative and amino acid stresses. The Pearson correlation coefficient is shown for each comparison.}
    \label{fig:stress_gasch_comparison}
% \end{figure}
    \vspace{1em}

% \begin{figure}[h]
%     \centering
    \includegraphics[width=6in]{figures/stress/stress_nitrogen_gene_ontology.pdf}
    \caption[Gene ontology terms enriched in genes with nitrogen-stress-induced genic TFIIB peaks]{Gene ontology terms enriched in genes with nitrogen-stress-induced genic TFIIB peaks.}
    \label{fig:stress_nitrogen_gene_ontology}
\end{figure}

\clearpage

\begin{figure}[h]
    \includegraphics[width=6in]{figures/stress/stress_tfiib_coverage.pdf}
    \caption[TFIIB ChIP-nexus protection over four genes with stress-induced intragenic TFIIB peaks.]{Relative TFIIB ChIP-nexus protection over four genes with an intragenic TFIIB peak significantly induced in one or more of the stress conditions. Relative protection indicates that the signal is independently scaled for each gene and cannot be compared between genes.}
    \label{fig:stress_tfiib_coverage}
\end{figure}

\clearpage

\begin{sidewaysfigure}
    \includegraphics[width=8.25in]{figures/stress/stress_tfiib_ridgelines.pdf}
    \caption[TFIIB ChIP-nexus protection over all genes with stress-induced intragenic TFIIB peaks.]{Relative TFIIB ChIP-nexus protection over all genes with an intragenic TFIIB peak significantly induced in one or more of the stress conditions tested, as depicted in the left panel. Genes are aligned by start codon, and are sorted within each group by the distance from the start codon to the summit of the induced intragenic TFIIB peak. Data are shown for each gene up to the stop codon of the gene. Regions where TFIIB peaks are called are shaded in the stress conditions according to the fold-change of the peak relative to the corresponding control condition.}
    \label{fig:stress_tfiib_ridgelines}
\end{sidewaysfigure}

\clearpage

\begin{SCfigure}[50][h]
    \includegraphics[width=4in]{figures/stress/stress_genic_vs_intra.pdf}
    \caption[Scatterplot of change in intragenic versus genic TFIIB ChIP-nexus signal, for all pairs of intragenic and genic TFIIB peaks in the three stress conditions.]{Scatterplot comparing change in intragenic TFIIB ChIP-nexus signal to the change in genic TFIIB signal at the same gene, for all pairs of intragenic and genic TFIIB peaks in the three stress conditions. Error bars indicate $\pm$ 1 standard error, and the Pearson correlation coefficient is shown.}
    \label{fig:stress_genic_vs_intra}
\end{SCfigure}

% \begin{SCfigure}[50][h]
\begin{wrapfigure}[10]{r}{3in}
    \includegraphics[width=3in]{figures/stress/stress_promoter_tss_diffexp_summary.pdf}
    \caption[Bar plot of the number of promoters in various genomic classes detected as differentially expressed in oxidative stress.]{Bar plot of the number of TSS peaks in various genomic classes that have corresponding TFIIB peaks and are detected as differentially expressed in oxidative stress.}
    \label{fig:stress_promoter_tss_diffexp_summary}
% \end{SCfigure}
\end{wrapfigure}
Because the greatest changes to intragenic transcription initiation were detected in oxidative stress, we focused on this condition and performed TSS-seq to determine which intragenic initiation events produce stable RNAs and in which strand orientation these events occur.
Considering only TSS peaks with a TFIIB peak overlapping the window extending 200 base pairs upstream of the TSS summit, we find cases of both sense intragenic and antisense TSSs that are differentially expressed in oxidative stress (Figure \ref{fig:stress_promoter_tss_diffexp_summary}).
In general, the induced intragenic TSSs we detect are expressed at lower levels than induced genic TSSs: The most abundant induced intragenic TSS is present in oxidative stress at levels comparable to the 54\textsuperscript{th} percentile of induced genic TSS abundances (Figure \ref{fig:stress_promoter_tss_expression}).

\begin{figure}[h]
    \includegraphics[width=6in]{figures/stress/stress_promoter_tss_expression.pdf}
    \caption[TSS-seq expression levels in oxidative stress of oxidative-stress-induced genic and intragenic promoters.]{Cumulative distributions of TSS-seq expression levels in oxidative stress, for all genic and intragenic TSS peaks that have corresponding TFIIB peaks and are significantly induced in oxidative stress. Error bars indicate $\pm$ one standard deviation.}
    \label{fig:stress_promoter_tss_expression}
\end{figure}

\section{Polysome enrichment of oxidative-stress-induced intragenic transcripts}

Translation of a transcript requires that the transcript possess both a 5$^\prime$-cap and a poly-A tail.
Since the TSS-seq protocol enriches for both of these features, this implies that the oxidative-stress-dependent intragenic transcripts we detect by TSS-seq could potentially be translated.
For sense-strand intragenic transcripts, this could generate N-terminally-truncated protein isoforms.
To see how likely oxidative-stress-dependent intragenic transcripts are to be targets for translation, we isolated polysomes and their associated RNAs by sucrose gradient fractionation, and sequenced TSS-seq libraries of the polysome-associated RNA.
Among oxidative-stress-induced TSSs with corresponding TFIIB peaks, intragenic TSSs in oxidative stress are less enriched in the polysome fraction compared to genic TSSs (Figure \ref{fig:stress_promoter_tss_polyenrichment}).
However, half of the induced intragenic TSSs are found in polysomes during oxidative stress at levels above the 25\textsuperscript{th} percentile of induced genic TSS levels in polysomes, indicating that many of the intragenic transcripts we identify are translated at some level.

\begin{figure}[h]
    \includegraphics[width=6in]{figures/stress/stress_promoter_tss_polyenrichment.pdf}
    \caption[Relative polysome enrichment in oxidative stress, for oxidative-stress-induced genic and intragenic promoters.]{Relative polysome enrichment in oxidative stress, for oxidative-stress-induced genic and intragenic TSS peaks with corresponding TFIIB peaks. Error bars indicate $\pm$ one standard error.}
    \label{fig:stress_promoter_tss_polyenrichment}
\end{figure}

\section{Functions of intragenic \textit{DSK2} expression in oxidative stress}

To investigate a particular case of oxidative-stress-induced intragenic transcription in more depth, we focused on the gene \textit{DSK2}, which is associated with the intragenic TSS we found to be most significantly induced upon oxidative stress.
The major \textit{DSK2} intragenic TSS is associated with a TFIIB peak that is similarly induced in oxidative stress (Figure \ref{fig:stress_tfiib_coverage}), and the intragenic transcript is also found in polysomes at intermediate levels (Figure \ref{fig:stress_promoter_tss_polyenrichment}).
Using a published dataset of histone modification MNase-ChIP-seq data during a timecourse of oxidative stress \citep{weiner2015}, we see that induction of intragenic \textit{DSK2} expression corresponds with shifts in transcription-associated histone modifications over the \textit{DSK2} gene, such as a spreading of H3K4me3 from the 5$^\prime$ end of the gene towards the 3$^\prime$ end (Figure \ref{fig:stress_dsk2_summary}).
\begin{figure}[h]
    \includegraphics[width=6in]{figures/stress/stress_dsk2_summary.pdf}
    \caption[Sense TSS-seq signal, TFIIB ChIP-nexus protection, and MNase-ChIP-seq data at the \textit{DSK2} gene, over an oxidative stress timecourse.]{Sense TSS-seq signal (black), TFIIB ChIP-nexus protection (gray), smoothed MNase-seq dyad signal, and relative H3K4me3 MNase-ChIP-seq enrichment over the gene \textit{DSK2}, over a timecourse of oxidative stress. MNase-seq and MNase-ChIP-seq data are from \citet{weiner2015}. The shaded bars on the \textit{DSK2} ORF indicate PACE-core elements on the sense (darker) or antisense (lighter) strands, which are potential binding sites for the transcription factor Rpn4 \citep{shirozu2015}. The lightly shaded region of the background indicates the boundaries of the intragenic TFIIB peak.}
    \label{fig:stress_dsk2_summary}
\end{figure}

\begin{wrapfigure}[9]{r}{3in}
    \centering
    \includegraphics[width=3in]{figures/stress/stress_dsk2_pace_northern.pdf}
    \caption[Northern blot for \textit{DSK2} transcripts in wild-type \textit{DSK2} and \textit{dsk2-pace} strains, in the absence or presence of oxidative stress.]{Northern blot for \textit{DSK2} transcripts in wild-type \textit{DSK2} and \textit{dsk2-pace} strains, in the absence or presence of oxidative stress induced by addition of diamide to the media. \textit{SNR190} is shown as a loading control.}
    \label{fig:stress_dsk2_pace_northern}
\end{wrapfigure}

Dsk2 is a member of a family of partially redundant ubiquitin receptors that shuttle polyubiquitinated proteins to the proteasome for degradation \citep{funakoshi2002}.
Upon inspection of the \textit{DSK2} DNA sequence, we discovered multiple PACE-core motifs within the \textit{DSK2} coding sequence, which are potential binding sites for the proteasome transcription factor Rpn4 \citep{shirozu2015}.
By making silent mutations to three PACE-core elements occurring just upstream of the \textit{DSK2} intragenic TFIIB peak (the more darkly shaded sense strand PACE-core elements in Figure \ref{fig:stress_dsk2_summary}, two of which are adjacent to each other), we generated a strain named \textit{dsk2-pace} in which intragenic \textit{DSK2} expression is eliminated (Figure \ref{fig:stress_dsk2_pace_northern}).
Full length \textit{DSK2} transcript levels in diamide are not noticeably affected in the \textit{dsk2-pace} mutant, suggesting that a primary function of intragenic \textit{DSK2} transcription is not likely to be control of full-length \textit{DSK2} transcript levels.
So far, our efforts to detect a possible intragenic Dsk2 protein have been inconclusive; if intragenic Dsk2 protein is produced, it may be highly unstable.

To test whether intragenic \textit{DSK2} expression is important for the cellular response to oxidative stress, we performed competitive growth experiments in which wild-type \textit{DSK2} and \textit{dsk2-pace} cells, distinguishable by expression of different fluorophores, were co-cultured during a timecourse of oxidative stress induced by addition of diamide to the media.
In these experiments, both strains contained deletions of \textit{RAD23}, another ubiquitin receptor that is partially redundant with \textit{DSK2} \citep{saeki2002, rao2002}.
At diamide concentrations around ~0.9-1.0 mM, we observe that \textit{dsk2-pace} cells begin to grow more slowly than wild-type \textit{DSK2} cells (Figure \ref{fig:stress_diamide_fitnesscomp}).
After two days in 1.25 mM diamide, \textit{dsk2-pace} cells make up less than 2\% of the co-culture, strongly suggesting that intragenic \textit{DSK2} expression does play a role in the response to oxidative stress.

\begin{figure}[h]
    \includegraphics[width=6in]{figures/stress/stress_diamide_fitnesscomp.pdf}
    \caption[Percentage of \textit{dsk2-pace} cells over two days of competitive growth against wild-type \textit{DSK2} cells at various concentrations of diamide.]{Percentage of \textit{dsk2-pace} cells over two days of competitive growth against wild-type \textit{DSK2} cells at various concentrations of diamide. The lines and shading are the mean $\pm$ one standard deviation of six replicates: Three in which \textit{dsk2-pace} and \textit{DSK2} cells are respectively marked by expression of mCherry and YFP, and three in which the fluorophores are swapped between strains.}
    \label{fig:stress_diamide_fitnesscomp}
\end{figure}

How might intragenic \textit{DSK2} expression function during oxidative stress?
Like other proteasome receptors, Dsk2 possesses an N-terminal ubiquitin-like domain which interacts with the proteasome, and a C-terminal ubiquitin-associating domain which binds polyubiquitin chains \citep{funakoshi2002}.
Translation of the intragenic \textit{DSK2} transcript is predicted to generate an intragenic protein lacking the\\ proteasome-interacting domain but retaining the polyubiquitin binding domain.
Such an intragenic protein could bind certain polyubiquitinated targets without delivering them to the proteasome, in effect preventing their degradation.
As noted above, we have so far been unable to reliably detect an intragenic Dsk2 protein; however, there is precedent for intragenic proteins to have different stability than their corresponding full-length forms \citep{gammie1999, benanti2009}.

Interestingly, one study has shown that an interaction between Dsk2 and an uncharacterized protein called Irc22 is involved in conferring salt tolerance in \textit{S. cerevisiae} \citep{ishii2014}.
This study showed that synthetic N-terminal truncations of Dsk2 missing the first 77 or 215 amino acids bound Irc22 more strongly than full-length Dsk2.
By comparison, the intragenic Dsk2 protein predicted to be expressed in oxidative stress would be missing the first 152 amino acids of Dsk2.

To further investigate the mode of intragenic \textit{DSK2} function, we are planning two additional fitness competition experiments.
In the first experiment, we will test whether intragenic \textit{DSK2} expression can complement in \textit{trans}, which should help differentiate whether it is intragenic \textit{DSK2} transcription, or the intragenic transcript or protein which provides protection from oxidative stress.
If intragenic \textit{DSK2} can be complemented in \textit{trans}, we will test whether intragenic \textit{DSK2} translation is required for protection from oxidative stress by mutation of intragenic start codons.

\section[TSS-seq analysis of oxidative stress\\ in \textit{Saccharomyces sensu stricto} species]{TSS-seq analysis of oxidative stress in \textit{Saccharomyces sensu stricto} species}

\begin{figure}[h]
    \includegraphics[width=6in]{figures/stress/stress_interyeast_intragenic.pdf}
    \caption[Heatmap of fold-change upon oxidative stress, for \textit{S. cerevisiae} intragenic TSSs differentially expressed in oxidative stress, and homologous TSSs found in \textit{S. mikatae} and \textit{S. bayanus}.]{Heatmap of fold-change upon oxidative stress, for all \textit{S. cerevisiae} intragenic TSSs differentially expressed in oxidative stress with corresponding TFIIB peaks, and homologous TSSs found in \textit{S. mikatae} and \textit{S. bayanus}. Gray tiles indicate that no homologous TSS was called in a species.}
    \label{fig:stress_interyeast_intragenic}
\end{figure}

If an instance of intragenic transcription like that at \textit{DSK2} is functional, then it might be conserved between different species.
To see if this is likely for the cases of oxidative-stress-dependent intragenic transcription we observe in \textit{S. cerevisiae}, we performed TSS-seq in oxidative stress for the related \textit{Saccharomyces sensu stricto} species \textit{Saccharomyces mikatae} and \textit{Saccharomyces bayanus}.
We then performed a multiple genome alignment of the three species, and determined which TSS peaks in \textit{S. cerevisiae} have corresponding TSSs called in the homologous region of the other species.
By TSS-seq, we do not find evidence of conservation for most oxidative-stress-dependent intragenic TSSs: Out of 39 differentially expressed intragenic \textit{S. cerevisiae} TSSs with matching TFIIB peaks, we find matching TSSs for only 5 TSSs in \textit{S. mikatae} (13\%) and 4 TSSs in \textit{S. bayanus} (10\%), compared to 42\% and 38\% of differentially expressed genic TSSs (Figure \ref{fig:stress_interyeast_intragenic}).
This is a low level of intragenic conservation when compared to previous reports of conservation of antisense transcripts \citep{yassour2010, rhind2011}, but may be consistent with the reported tendency of long noncoding RNAs to be gained and lost at a rapid rate compared to protein-coding genes \citep{kutter2012}.

Interestingly, the gene \textit{GRX2}, whose intragenic transcript is induced upon oxidative stress in all three species, is known to express two protein isoforms from different start codons \citep{pedrajas2002}.
These two isoforms localize to different subcellular compartments, and our data suggest intragenic transcription may explain the use of the downstream start codon to express the shorter isoform.

\begin{figure}[h]
    \includegraphics[width=6in]{figures/stress/stress_dsk2_interyeast.pdf}
    \caption[Sense TSS-seq signal over the \textit{DSK2} gene in \textit{S. cerevisiae, S. mikatae, and S. bayanus} in unstressed and oxidative stress conditions.]{Relative sense TSS-seq signal over the \textit{DSK2} gene in \textit{S. cerevisiae, S. mikatae, and S. bayanus} in unstressed and oxidative stress conditions. The shaded bars on the \textit{DSK2} gene diagrams indicate PACE-core elements on the sense (darker) or antisense (lighter) strands. The lightly shaded regions in the background indicate the boundaries of regions homologous to the two \textit{DSK2} intragenic TSS-seq peaks called in \textit{S. cerevisiae}, for each species.}
    \label{fig:stress_dsk2_interyeast}
\end{figure}

For \textit{DSK2}, the three PACE-core elements needed for intragenic \textit{DSK2} expression in \textit{S. cerevisiae} are conserved in \textit{S. mikatae} but not \textit{S. bayanus} (Figure \ref{fig:stress_dsk2_interyeast}).
Despite this, we do not observe the intragenic \textit{DSK2} TSS-seq peak in \textit{S. mikatae}, with only a miniscule amount of TSS-seq signal over the region homologous to the \textit{S. cerevisiae} peak.
Consistent with this, when probing for \textit{DSK2} transcripts in the three species by Northern blot, we observe extremely low expression of intragenic \textit{DSK2} RNA in \textit{S. mikatae} upon oxidative stress and no detectable intragenic \textit{DSK2} RNA in \textit{S. bayanus} (Figure \ref{fig:stress_dsk2_interyeast_northern}).

\begin{figure}[h]
    \centering
    \includegraphics[width=5in]{figures/stress/stress_dsk2_interyeast_northern.pdf}
    \caption[Northern blot for \textit{DSK2} transcripts in the yeasts \textit{S. cerevisiae, S. mikatae, and S. bayanus}, in the absence and presence of oxidative stress.]{Northern blot for \textit{DSK2} transcripts in the yeasts \textit{S. cerevisiae, S. mikatae, and S. bayanus}, in the absence and presence of oxidative stress induced by addition of diamide to the media. \textit{HSP12} is shown as a positive control for oxidative stress, and \textit{SNR190} is shown as a loading control.}
    \label{fig:stress_dsk2_interyeast_northern}
\end{figure}

\section{Discussion}

In this work, we used high resolution genomic assays of transcription initiation to identify intragenic initiation regulated by three stress conditions.
For the condition of oxidative stress, we show that intragenic transcripts are generally expressed at lower levels than genic transcripts, and many are likely to be translated at some level.
Most of the oxidative-stress-regulated transcripts we identify are not conserved among three \textit{Saccharomyces sensu stricto} species, which may suggest that intragenic promoters evolve very rapidly.
At least one instance of oxidative-stress-induced intragenic transcription, at the gene \textit{DSK2}, is important for yeast to survive under conditions of oxidative stress.
Our ongoing experiments are aimed at determining how intragenic \textit{DSK2} functions to confer oxidative stress resistance.
Further studies of examples of intragenic transcription like the ones identified in this study are likely to illuminate many ways in which intragenic transcription functions within cells.

\section{Methods}

\subsection{Yeast growth conditions}
\label{subsec:stress_growth_conditions}

For oxidative stress studies in all three yeast species, yeast were grown at 30\textdegree C in YPD rich medium to mid-log phase (OD\textsubscript{600} $\sim$ 0.4), at which point diamide was added to a final concentration of 1.5 mM.
Samples were collected after 30 (\textit{S. bayanus}) or 45 minutes (\textit{S. cerevisiae} and \textit{S. mikatae}) of diamide treatment, timepoints which were chosen based on induction of the \textit{HSP12} gene.

For amino acid stress, \textit{S. cerevisiae} strain FY3126 was grown at 30\textdegree C in SC synthetic complete medium to mid-log phase (OD\textsubscript{600} $\sim$ 0.4), collected by centrifugation, washed with SD minimal medium supplemented with uracil and lysine (SD+Ura+Lys; 0.15\% yeast nitrogen base, 0.5\% ammonium sulfate, 2\% glucose, 20 mg/L uracil, 0.004\% lysine), resuspended in an equal volume of SD+Ura+Lys, and harvested after 30 minutes of starvation.

For nitrogen stress, \textit{S. cerevisiae} strain FY3126 was grown at 30\textdegree C in SC to mid-log phase (OD\textsubscript{600} $\sim$ 0.4), collected by centrifugation, washed with nitrogen deficient medium (YNB-AA-AS; 0.0025\% ammonium sulfate, 2\% glucose, 20 mg/L uracil, 0.004\% lysine), and resuspended in an equal volume of nitrogen deficient medium, and harvested after 9.5 hours of starvation.

\subsection{Sequencing library preparation (TFIIB ChIP-nexus, TSS-seq)}

TFIIB ChIP-nexus libraries and all TSS-seq libraries for all species, including polysome RNA TSS-seq libraries, were prepared as described in \citet{doris2018}.
\textit{S. pombe} cells were used as a spike-in for all TSS-seq libraries except for polysome RNA TSS-seq libraries.

\subsection{Genome builds}

The genome build used for \textit{S. cerevisiae} was R64-2-1 \citep{engel2014}, and the genome build used for \textit{S. pombe} spike-ins was ASM294v2 \citep{wood2002}.
For \textit{S. mikatae }IFO 1815\textsuperscript{T} and \textit{S. bayanus} var. \textit{uvarum }CBS 7001, ultra-scaffold sequences from \citet{scannell11} were used as genomes.

\subsection{TFIIB ChIP-nexus data analysis}

TFIIB ChIP-nexus data analyses were carried out as described in section \ref{subsec:chipnexus}.

\subsection{TSS-seq data analysis}

TSS-seq data analyses were carried out as described in section \ref{subsec:tss_seq}, except using the relevant genomes for \textit{S. mikatae} and \textit{S. bayanus} libraries.

For \textit{S. mikatae} and \textit{S. bayanus}, annotation of the 5$^\prime$ ends of transcripts was performed as described in section \ref{subsubsec:tss_reannotation}, using ORF annotation GFFs from \citet{scannell11} as both transcript and ORF annotations, and two replicates of unstressed wild-type TSS-seq data.
The 5$^\prime$ end of an annotation was adjusted if it was above the 90\textsuperscript{th} (\textit{S. mikatae}) or 85\textsuperscript{th} (\textit{S. bayanus}) percentile of all non-zero TSS-seq signal.

\subsection{Sucrose gradient fractionation}

\textit{S. cerevisiae} strain FY3126 was grown in 250 mL cultures in YPD or YPD with diamide \hyperref[subsec:stress_growth_conditions]{as described above}.
Prior to harvesting, cells were treated with 0.1 mg/mL cycloheximide and incubated for 5 minutes at 30\textdegree C.
Cell pellets were resuspended in 0.5 mL ice-cold polysome lysis buffer (20 mM Tris-HCl pH 8.0, 140 mM KCl, 1.5 mM MgCl\textsubscript{2}, 1\% Triton X-100, 100 \textmu g/mL cycloheximide), and lysed by bead beating as previously described \citep{degennaro2013}.
Lysates containing approximately 50 A\textsubscript{260} absorbance units were layered on top of 10-50\% (weight/volume) sucrose gradients (20 mM Tris-HCl pH 8.0, 140 mM KCl, 5 mM MgCl\textsubscript{2}, 100 \textmu g/mL cycloheximide) made using a Biocomp Instruments gradient master.
Gradients were centrifuged for 2 hours at 36,000 rpm in an SW-41 rotor and fractioned with a Brandel gradient fractionator and ISCO UA-6 detector.
RNA from fractions 5-12 were pooled and used to construct TSS-seq libraries.

\subsection{Polysome RNA TSS-seq analysis}

Polysome RNA TSS-seq libraries were processed in the same manner as total RNA TSS-seq libraries, except no spike-in normalization was performed because no spike-in was included.
A separate Snakemake pipeline was used to integrate data from total RNA and polysome RNA TSS-seq.
This pipeline performs three differential expression analyses using DESeq2 \citep{love2014}: Two analyses compare polysome RNA to total RNA in either the experimental condition (i.e., oxidative stress) or the control condition (i.e., YPD) in order to estimate relative polysome enrichment,\\ $\log_2 \frac{\text{polysome RNA}}{\text{total RNA}}$.
Polysome enrichment is a relative measure of how much more or less polysome RNA is detected than is expected, given the amount of total RNA detected.
The third differential expression analysis tests whether polysome enrichment changes between the control condition and the experimental condition, i.e., whether the experimental perturbation caused changes to the relative association of a transcript with polysomes.
The pipeline also finds all potential ORFs downstream of intragenic TSSs.

\subsection{MNase-ChIP-seq data analysis}

FASTQ files of the MNase-ChIP-seq data from \citet{weiner2015} were obtained from SRA (\href{https://www.ncbi.nlm.nih.gov/sra?term=SRP048526}{SRP048526}), encompassing 201 libraries for ChIP of 26 histone modifications over 6 timepoints with inputs (i.e., MNase-seq libraries).
A Snakemake pipeline was created to process this data.
Reads were cleaned by 3$^\prime$ quality trimming and adapter removal using cutadapt \citep{martin2011}.
Reads were aligned to the \textit{S. cerevisiae} genome using Bowtie 2 \citep{langmead2012}, and uniquely mapping alignments were selected using SAMtools \citep{li2009}.
The median fragment size estimated by MACS2 \citep{zhang2008} over all samples was used to generate coverage of factor protection and fragment midpoints, by extending reads to the fragment size, or by shifting reads by half the fragment size, respectively.
Smoothed nucleosome dyad coverage was generated by smoothing fragment midpoint coverage with a Gaussian kernel of 20 bp bandwidth.
Coverage was normalized to the total number of reads uniquely mapping to the genome.
Relative enrichment of histone modifications was calculated as a normalized log-ratio of IP coverage over input (MNase-seq) coverage.

\subsection{Multiple genome alignment}

A Snakemake pipeline was created which uses progressiveMauve \citep{darling2010} to perform multiple genome alignment.
The multiple genome alignment is used to translate annotations such as TSSs called in one species to homologous coordinates in another species, and to extract and visualize data coverage over homologous regions in multiple species.

A separate Snakemake pipeline was created to find homologous TSSs between the three species by looking for overlap between \textit{S. cerevisiae} TSS peaks and peaks called in the other species which were translated into \textit{S. cerevisiae} coordinates.

\subsection{Diamide competitive fitness assays}

For diamide competitive fitness assays, overnight cultures grown in YPD were diluted and allowed to grow to mid-log phase in YPD.
YPD was then added to each culture as needed to make the OD\textsubscript{600} readings of all cultures equivalent.
To mix competing strains, 150 \textmu L of each strain was mixed by pipetting in a 96-well plate.
Then, 6 \textmu L of each cell mixture was used to inoculate 600 \textmu L of YPD or YPD+diamide in a 96-well assay block.
Prior to culture growth, an initial cell count was performed by flow cytometry of 100 \textmu L of the culture in 100 \textmu L of TE, using a Stratedigm S1000EX flow cytometer.
Cultures were then allowed to grow at 30\textdegree C with shaking.
Every 24 hours, cell counting was performed by flow cytometry of 5 \textmu L of the culture in 195 \textmu L of TE, cultures were diluted 1:100 to a total volume of 600 \textmu L in YPD or YPD+diamide, and the diluted cultures were returned to 30\textdegree C with shaking.
For flow cytometry, YFP fluorescence was detected using 488 nm excitation with a 530/30 nm filter, and mCherry fluorescence was detected using 552 nm excitation with a 615/30 nm filter.

Flow cytometry events were labeled positive for YFP if the YFP signal was at least 500 arbitrary units, and positive for mCherry if the mCherry signal was at least 250 arbitrary units.
Double positive and double negative events were excluded from analysis.

\subsection{Northern blotting}

Northern blotting was performed as described in \citet{degennaro2013}.

\newpage
\bibliographystyle{apalike}
\begingroup
\singlespacing
\bibliography{references/stress}
\endgroup
